<!doctype html><html><head><meta charset=UTF-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=generator content="Hugo 0.146.6"><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="List of SBEnv"><title>List of Software in SBEnv environment | Grex</title>
<link rel=icon href=/grex-docs/logo/um_logo_vertical.png type=image/x-icon><link rel=stylesheet rel=preload type=text/css href=/grex-docs/css/main.css><link rel=stylesheet type=text/css href=/grex-docs/css/external/fontawesome.all.min.css><link rel=stylesheet type=text/css href></head><body onload=browserCompatibility()><div id=navbarInfo class="is-hidden modal-container"><div class="modal-content-box modal-text-box"><div class="card modal-title is-radiusless"><div class=card-header><p class="card-header-title is-marginless">About</p></div></div><div class=modal-content-container><div class="modal-content-block is-block"><div class='columns is-marginless is-multiline is-centered'><div class=column><a class=is-itext-link href=https://github.com/um-grex/grex-docs target=_blank><div><i class='fa-brands fa-github fa-2x'></i><p>GitHub repository</p></div></a></div><div class=column><a class=is-itext-link href=https://github.com/um-grex/grex-docs/archive/refs/heads/main.zip target=_blank><div><i class='fa-solid fa-download fa-2x'></i><p>Download</p></div></a></div><div class=column><a class=is-itext-link href=https://github.com/um-grex/grex-docs/issues target=_blank><div><i class='fa-solid fa-circle-info fa-2x'></i><p>Report an issue</p></div></a></div><div class=column><a class=is-itext-link href=https://github.com/um-grex/grex-docs/forks target=_blank><div><i class='fa-solid fa-code-branch fa-2x'></i><p>Fork</p></div></a></div></div><p>Website built with <a href=https://gohugo.io/ class=is-pretty-link>Hugo</a> (0.146.6)</p><p>The MIT License (MIT) Copyright © 2023 UM-Grex ()</p></div></div></div></div><div id=navbarShortcuts class="is-hidden modal-container"><div class="modal-content-box modal-text-box"><div class="card modal-title is-radiusless"><div class=card-header><p class="card-header-title is-marginless">Keyboard shortcuts</p></div></div><div class=modal-content-container><div class=modal-content-block><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>q</kbd></div><div class=modal-content><p class=has-text-weight-normal>Show current page QR code</p></div><div class=modal-content><kbd>Escape</kbd></div><div class=modal-content><p class=has-text-weight-normal>Close modals</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>i</kbd></div><div class=modal-content><p class=has-text-weight-normal>Show website information</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>k</kbd></div><div class=modal-content><p class=has-text-weight-normal>Show shortcuts</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>h</kbd></div><div class=modal-content><p class=has-text-weight-normal>Go to homepage</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>f</kbd></div><div class=modal-content><p class=has-text-weight-normal>Find on website</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>m</kbd></div><div class=modal-content><p class=has-text-weight-normal>Collapse/Uncollapse sidebar</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>t</kbd></div><div class=modal-content><p class=has-text-weight-normal>Collapse/Uncollapse table of contents</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>↑</kbd></div><div class=modal-content><p class=has-text-weight-normal>Go to the top of the page</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>↓</kbd></div><div class=modal-content><p class=has-text-weight-normal>Go to the bottom of the page</p></div><div class=modal-content><kbd>Shift</kbd><p>+</p><kbd>p</kbd></div><div class=modal-content><p class=has-text-weight-normal>Print current page</p></div></div></div></div></div><div id=modalContainer class="is-hidden modal-container"><div id=modal class=modal-content-box></div></div><nav id=navbar class=navbar role=navigation aria-label="main navigation"><div id=navbarItemsContainer class=navbar-brand><div id=globalLogoContainer class=is-flex><a id=globalLogo class="navbar-item navbar-item-standard is-paddingless" href=/grex-docs/><img alt=Homepage src=/grex-docs/logo/um_logo_email_signature.png>
</a><a id=globalTouchLogo class="navbar-item navbar-item-standard is-paddingless" href=/grex-docs/><img alt=Homepage src=/grex-docs/logo/um_logo_email_signature.png></a></div><div id=navbarItems class=is-flex><div id=navbarItemsStart class=is-flex><div id=searchContainer class=navbar-item><div class="control has-icons-left"><span id=searchInput class=autocomplete><input id=search autocomplete=off class=input type=search placeholder=Search>
</span><span class="icon is-left"><i class="fa-solid fa-magnifying-glass"></i></span><ul id=searchList class="is-paddingless is-hidden"></ul></div></div></div><div id=navbarItemsEnd class="is-flex is-invisible"><div class=navbar-item><a id=printButton class="button navbar-item-standard"><span class=icon-text>Print</span>
<span class=icon><i class="fa-solid fa-print fa-lg"></i></span></a></div><div class=navbar-item><a id=qrCodeButton class="button navbar-item-standard"><span class=icon-text>QR code</span>
<span class=icon><i class="fa-solid fa-qrcode fa-lg"></i></span></a></div><div class=navbar-item><a id=shortcutsInfo trigger=navbarShortcuts class="button navbar-item-standard navbar-trigger" title="Keyboard shortcuts"><span class=icon-text>Shortcuts</span>
<span class=icon><i class="fa-solid fa-keyboard fa-lg"></i></span></a></div><div class=navbar-item><div id=taxonomiesSelectorContainer class="dropdown is-right"><div class=dropdown-trigger><button id=taxonomiesSelector class="button navbar-item-standard" aria-haspopup=true aria-controls=dropdown-menu>
<span class=icon-text>Taxonomies</span>
<span class=icon><i class="fa-solid fa-tags fa-lg"></i></span></button></div><div id=dropdown-menu-taxonomies class=dropdown-menu role=menu><div class="dropdown-content is-paddingless"><a href=/grex-docs/categories/ class="dropdown-item has-text-weight-bold">Categories</a><a href=/grex-docs/tags/ class="dropdown-item has-text-weight-bold">Tags</a></div></div></div></div><div class=navbar-item><a id=siteInfo trigger=navbarInfo class="button navbar-item-standard navbar-trigger" title=About><span class=icon-text>About</span>
<span class=icon><i class="fa-solid fa-circle-question fa-lg"></i></span></a></div><div id=navbarExtend class=navbar-item><div id=navbarExtendWrapper><a id=navbarExtendButton class="button navbar-item-standard"><span class=icon><i class="fa-solid fa-ellipsis fa-lg"></i></span></a><div id=navbarExtendItemsContainer><div id=navbarExtendItemsWrapper><div class=navbar-item><a id=printButtonExtend class="button navbar-item-standard"><span class=icon-text>Print</span>
<span class=icon><i class="fa-solid fa-print fa-lg"></i></span></a></div><div class=navbar-item><a id=qrCodeButtonExtend class="button navbar-item-standard"><span class=icon-text>QR code</span>
<span class=icon><i class="fa-solid fa-qrcode fa-lg"></i></span></a></div><div class=navbar-item><a id=shortcutsInfoExtend trigger=navbarShortcuts class="button navbar-item-standard navbar-trigger" title="Keyboard shortcuts"><span class=icon-text>Shortcuts</span>
<span class=icon><i class="fa-solid fa-keyboard fa-lg"></i></span></a></div><div class=navbar-item><div id=taxonomiesSelectorContainerExtend class="dropdown is-right"><div class=dropdown-trigger><button id=taxonomiesSelectorExtend class="button navbar-item-standard" aria-haspopup=true aria-controls=dropdown-menu>
<span class=icon-text>Taxonomies</span>
<span class=icon><i class="fa-solid fa-tags fa-lg"></i></span></button></div><div id=dropdown-menu-taxonomiesExtend class=dropdown-menu role=menu><div class="dropdown-content is-paddingless"><a href=/grex-docs/categories/ class="dropdown-item has-text-weight-bold">Categories</a><a href=/grex-docs/tags/ class="dropdown-item has-text-weight-bold">Tags</a></div></div></div></div><div class=navbar-item><a id=siteInfoExtend trigger=navbarInfo class="button navbar-item-standard navbar-trigger" title=About><span class=icon-text>About</span>
<span class=icon><i class="fa-solid fa-circle-question fa-lg"></i></span></a></div></div></div></div></div></div></div></div></nav><div id=navbarOverlay></div><div class="columns is-mobile is-paddingless is-marginless" id=mainContainer><div id=sidebarContainer class="column is-narrow is-paddingless is-marginless"><div id=sidebarWrapper class=is-fixed><div id=sidebar class=is-marginless><aside class="menu is-paddingless is-marginless"><ul class="menu-list is-marginless is-paddingless"><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/updates/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/updates/ class=card-header-title title='Grex Updates'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Grex Updates</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/grex/ class=card-header-title><i class='fa-solid fa-user-gear fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/grex/ class=card-header-title title=Home><i class='fa-solid fa-user-gear fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Home</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/start-guide/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/start-guide/ class=card-header-title title='Quick Start Guide'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Quick Start Guide</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/access/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/access/ class=card-header-title title='Access and usage conditions'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Access and usage conditions</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/connecting/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/connecting/ class=card-header-title title='Connect / Transfer data'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Connect / Transfer data</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/connecting/mfa/ class=card-header-title title='Multi-Factor Authentication'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Multi-Factor Authentication</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/connecting/ssh/ class=card-header-title title='Connecting with SSH'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Connecting with SSH</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/connecting/ood/ class=card-header-title title='Connect with OOD'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Connect with OOD</p></a></div></div></li></div></div><div><div><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/connecting/data-transfer/ class=card-header-title title='Data transfer'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Data transfer</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/connecting/data-transfer/globus/ class=card-header-title title='Globus on Grex '><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Globus on Grex</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/connecting/data-transfer/one-drive/ class=card-header-title title='Rclone and OneDrive'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Rclone and OneDrive</p></a></div></div></li></div></div></ul></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/storage/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/storage/ class=card-header-title title='Storage and Data'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Storage and Data</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/storage/data-sizes-and-quota/ class=card-header-title title='Data sizes and Quotas'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Data sizes and Quotas</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/storage/data-backup/ class=card-header-title title='Data Backup'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Data Backup</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/storage/data-sharing/ class=card-header-title title='Data sharing'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Data sharing</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/running-jobs/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/running-jobs/ class=card-header-title title='Running jobs'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Running jobs</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/running-jobs/slurm-partitions/ class=card-header-title title='Slurm partitions'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Slurm partitions</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/running-jobs/interactive-jobs/ class=card-header-title title='Interactive jobs'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Interactive jobs</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/running-jobs/batch-jobs/ class=card-header-title title='Batch jobs'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Batch jobs</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/running-jobs/contributed-systems/ class=card-header-title title='Contributed nodes'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Contributed nodes</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/running-jobs/using-localdisks/ class=card-header-title title='Using local disks'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Using local disks</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon is-sidebar-active"><div class=card-header><a href=/grex-docs/software/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-tree-active is-entries-expandable is-entries-expanded"><div class=card><div class=card-header><a href=/grex-docs/software/ class=card-header-title title='Software / Applications'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Software / Applications</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-expanded"><span class=icon></span></a></div></div><ul class="is-marginless is-tree-active"><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/software/general-linux/ class=card-header-title title='Linux tools'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Linux tools</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/software/using-modules/ class=card-header-title title='Using modules'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Using modules</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/software/code-development/ class=card-header-title title='Code development'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Code development</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/software/containers/ class=card-header-title title=Containers><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Containers</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/software/cern-vmfs/ class=card-header-title title=CVMFS><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">CVMFS</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class="card is-sidebar-active" id=sidebarActiveEntry><div class="card-header is-single-link"><a href=/grex-docs/software/software-list/ class=card-header-title title='List of Software'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">List of Software</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/specific-soft/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/specific-soft/ class=card-header-title title='Software Specific Notes'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Software Specific Notes</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/espresso/ class=card-header-title title=Espresso><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Espresso</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/gaussian/ class=card-header-title title=Gaussian><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Gaussian</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/gromacs/ class=card-header-title title=GROMACS><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">GROMACS</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/julia/ class=card-header-title title=Julia><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Julia</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/jupyter-notebook/ class=card-header-title title='Jupyter notebooks'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Jupyter notebooks</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/lammps/ class=card-header-title title=LAMMPS><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">LAMMPS</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/matlab/ class=card-header-title title=MATLAB><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">MATLAB</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/nwchem/ class=card-header-title title=NWChem><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">NWChem</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/orca/ class=card-header-title title=ORCA><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">ORCA</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/vasp/ class=card-header-title title=VASP><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">VASP</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/specific-soft/python-ai/ class=card-header-title title='Python for ML'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Python for ML</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/ood/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/ood/ class=card-header-title title=OpenOnDemand><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">OpenOnDemand</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/changes/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/changes/ class=card-header-title title='Grex changes'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Grex changes</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/changes/linux-slurm-update/ class=card-header-title title='Dec 10-11, 2019'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Dec 10-11, 2019</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/changes/changes-before-2020/ class=card-header-title title='Before Dec, 2019'><i class='fa-solid fa-turn-up fa-rotate-90 fa-xs'></i><p class="pt-0 pb-0 pr-2 pl-0">Before Dec, 2019</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/support/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/support/ class=card-header-title title='Getting Help'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Getting Help</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/training/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/training/ class=card-header-title title=Workshops><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Workshops</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/training/workshops-2024/ class=card-header-title title=2024><i class='fa-solid fa-cubes'></i><p class="pt-0 pb-0 pr-2 pl-0">2024</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/training/workshops-2023/ class=card-header-title title=2023><i class='fa-solid fa-cubes'></i><p class="pt-0 pb-0 pr-2 pl-0">2023</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/training/workshops-2022/ class=card-header-title title=2022><i class='fa-solid fa-cubes'></i><p class="pt-0 pb-0 pr-2 pl-0">2022</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/training/workshops-2021/ class=card-header-title title=2021><i class='fa-solid fa-cubes'></i><p class="pt-0 pb-0 pr-2 pl-0">2021</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class=card-header><a href=/grex-docs/friends/ class=card-header-title><i class='fa-solid fa-house-chimney fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class="is-marginless is-entries-expandable is-entries-shrinked"><div class=card><div class=card-header><a href=/grex-docs/friends/ class=card-header-title title='Friendly Organizations'><i class='fa-solid fa-house-chimney fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Friendly Organizations</p></a><a class="card-header-icon is-icon-expandable is-paddingless is-icon-shrinked"><span class=icon></span></a></div></div><ul class=is-marginless><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/friends/alliancecan/ class=card-header-title title='DRAC (Alliance)'><i class='fa-solid fa-house-chimney'></i><p class="pt-0 pb-0 pr-2 pl-0">DRAC (Alliance)</p></a></div></div></li></div></div><div><div><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/friends/localit/ class=card-header-title title='UManitoba IT resources'><i class='fa-solid fa-house-chimney'></i><p class="pt-0 pb-0 pr-2 pl-0">UManitoba IT resources</p></a></div></div></li></div></div></ul></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/faq/ class=card-header-title><i class='fa-solid fa-circle-question fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/faq/ class=card-header-title title=FAQ><i class='fa-solid fa-circle-question fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">FAQ</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/deprecated/ class=card-header-title><i class='fa-solid fa-ban fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/deprecated/ class=card-header-title title='Deprecated pages'><i class='fa-solid fa-ban fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Deprecated pages</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/disclaimer/ class=card-header-title><i class='fa-solid fa-ban fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/disclaimer/ class=card-header-title title=Disclaimer><i class='fa-solid fa-ban fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Disclaimer</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/glossary/ class=card-header-title><i class='fa-solid fa-sitemap fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/glossary/ class=card-header-title title=Glossary><i class='fa-solid fa-sitemap fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Glossary</p></a></div></div></li></div></div><div class="card is-fs-expandable-icon"><div class="card-header is-single-link"><a href=/grex-docs/sitemap/ class=card-header-title><i class='fa-solid fa-sitemap fa-lg'></i></a></div></div><div class=is-expandable><div class=is-sidebar-list-wrapper><li class=is-marginless><div class=card><div class="card-header is-single-link"><a href=/grex-docs/sitemap/ class=card-header-title title=Sitemap><i class='fa-solid fa-sitemap fa-lg'></i><p class="pt-0 pb-0 pr-2 pl-1">Sitemap</p></a></div></div></li></div></div></ul></aside></div><div id=sidebarCollapsible class=is-marginless><div class="card is-uncollapse-action"><div id=sidebarUncollapse class="card-header is-single-link" title="Collapse/Uncollapse sidebar"><a class=card-header-title><i class="fa-solid fa-angles-right fa-lg"></i></a></div></div><div class="card is-collapse-action"><div id=sidebarCollapse class="card-header is-single-link" title="Collapse/Uncollapse sidebar"><a class=card-header-title><i class="fa-solid fa-angles-left fa-lg"></i><p>Collapse sidebar</p></a></div></div></div></div></div><div id=sidebarMobileWrapper class="column is-narrow is-hidden-desktop"></div><div class="columns is-mobile is-marginless is-scroll-smooth has-toc" id=contentContainer><div class=column id=content><div id=contentTitle>List of Software in SBEnv environment</div><h2 id=software-available-on-the-ccenv-software-stack-on-grex>Software available on the CCEnv software stack on Grex.<a href=#software-available-on-the-ccenv-software-stack-on-grex class=anchor>#</a></h2><p>Grex receives the ComputeCanada/Alliance software environment via CVMFS. Full autogenerated list of software is available at the <a href=https://docs.alliancecan.ca/wiki/Available_software class=is-pretty-link>Available Software page</a>
on the Allince Documentation Wiki.</p><blockquote><p>Note that Grex does not install local, commercial software licensed only to the National/Alliance systems. The software listed under &ldquo;List of globally-installed modules&rdquo; is available.</p></blockquote><h2 id=software-available-on-the-sbenv-software-stack-on-grex>Software available on the SBEnv software stack on Grex.<a href=#software-available-on-the-sbenv-software-stack-on-grex class=anchor>#</a></h2><p>Below is the auto-generated list of software modules available on Grex&rsquo;s local SBEnv software stack.</p><blockquote><p>Note that many items on this list, most notably for commercial software, are available only to the licensed software owners/research groups of that software! Please use <strong>module spider</strong> , <strong>module help</strong> and <strong>module load</strong> commands to check if a particular software module can be loaded by your group.</p></blockquote><p>The list is updated periodically. The most authoritative method to check for available software is still <strong>module spider</strong> as described on the Using Modules page.</p><table><thead><tr><th>Software</th><th>Versions</th><th>Description / Homepage</th><th>Module dependencies</th></tr></thead><tbody><tr><td>CCEnv</td><td></td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for CCEnv</div><div class=sc-collapsible-content>Description: Compute Canada software modules. Must be loaded to see Compute Canada modules in &lsquo;module spider&rsquo;</div></div><a href=https://docs.computecanada.ca/wiki/Available_software class=is-pretty-link>https://docs.computecanada.ca/wiki/Available_software</a></td><td></td></tr><tr><td>SBEnv</td><td></td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for SBEnv</div><div class=sc-collapsible-content>Alma8 software environment for Grex usin Niagara SB</div></div></td><td></td></tr><tr><td>adf</td><td>2021.106-impi, 2019.305-impi, 2024.105-impi-aocl, 2024.105-impi, 2023.104-impi, 2021.107-impi</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for adf</div><div class=sc-collapsible-content>AMS, Amsterdam Density Functional suite of codes Georg Schrecnekbach group at UManitoba, IntelMPI version</div></div></td><td></td></tr><tr><td>admixture</td><td>1.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for admixture</div><div class=sc-collapsible-content>ADMIXTURE: Fast ancestry estimation</div></div></td><td></td></tr><tr><td>amber</td><td>24</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for amber</div><div class=sc-collapsible-content>Description: Amber (originally Assisted Model Building with Energy Refinement) is software for performing molecular dynamics and structure prediction</div></div><a href=https://ambermd.org/ class=is-pretty-link>https://ambermd.org/</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>ansys</td><td>2023R2, 21.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ansys</div><div class=sc-collapsible-content>ANSYS suite of engineering simulation software</div></div></td><td></td></tr><tr><td>ant</td><td>1.10.15</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ant</div><div class=sc-collapsible-content>Description: Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications</div></div><a href=https://ant.apache.org/ class=is-pretty-link>https://ant.apache.org/</a></td><td></td></tr><tr><td>aocc</td><td>4.2.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for aocc</div><div class=sc-collapsible-content>Description: The AOCC compiler system is a high performance, production quality code generation tool</div></div><a href=https://developer.amd.com/amd-aocc/ class=is-pretty-link>https://developer.amd.com/amd-aocc/</a></td><td>( arch/avx2 ), ( arch/avx512 )</td></tr><tr><td>aocl</td><td>4.2.0, 4.2.0-64</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for aocl</div><div class=sc-collapsible-content>Description: AMD Optimizing CPU Libraries (AOCL) are a set of numerical libraries optimized for AMD EPYC processor family</div></div><a href=https://developer.amd.com/amd-aocl/ class=is-pretty-link>https://developer.amd.com/amd-aocl/</a></td><td>( arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 aocc/4.2.0 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 )</td></tr><tr><td>arch</td><td>avx2, avx512</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for arch</div><div class=sc-collapsible-content>Description: a module to pick the CPU architecture</div></div></td><td>( cuda/11.8.0 ), ( cuda/12.2.2 ), ( cuda/12.4.1 ),</td></tr><tr><td>armadillo</td><td>11.4.3, 14.2.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for armadillo</div><div class=sc-collapsible-content>; Description: Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions</div></div><a href=http://arma.sourceforge.net class=is-pretty-link>http://arma.sourceforge.net</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>arpack-ng</td><td>3.9.1+mkl-2019.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for arpack-ng</div><div class=sc-collapsible-content>Description: ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems</div></div><a href=https://github.com/opencollab/arpack-ng class=is-pretty-link>https://github.com/opencollab/arpack-ng</a></td><td>( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>arrow</td><td>18.1.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for arrow</div><div class=sc-collapsible-content>Description: Apache Arrow is a cross-language development platform for in-memory data</div></div><a href=https://arrow.apache.org class=is-pretty-link>https://arrow.apache.org</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>asynch</td><td>git-a5d1d77, 1.4.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for asynch</div><div class=sc-collapsible-content>Asynch is numerical library for solving differential equations with a tree structure</div></div></td><td>( arch/avx512 gcc/13.2.0 openmpi/5.0.6 )</td></tr><tr><td>autodock</td><td>4.2.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for autodock</div><div class=sc-collapsible-content>Description: AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure</div></div><a href=http://autodock.scripps.edu/ class=is-pretty-link>http://autodock.scripps.edu/</a></td><td>( arch/avx512 gcc/13.2.0 ), ( arch/avx512 intel/2023.2 )</td></tr><tr><td>autodock-vina</td><td>1.2.7</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for autodock-vina</div><div class=sc-collapsible-content>Description: AutoDock Vina is an open-source program for doing molecular docking</div></div><a href=http://vina.scripps.edu/index.html class=is-pretty-link>http://vina.scripps.edu/index.html</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>autotools</td><td>2022a</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for autotools</div><div class=sc-collapsible-content>Description: GNU autoconf, automake, gettext, and libtool</div></div><a href=https://www.gnu.org/software/%7bautoconf,automake,gettext,libtool%7d/ class=is-pretty-link>https://www.gnu.org/software/{autoconf,automake,gettext,libtool}/</a></td><td>( arch/avx512 ), , ( arch/avx2 )</td></tr><tr><td>bamtools</td><td>2.5.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for bamtools</div><div class=sc-collapsible-content>Description: BamTools provides both a programmer&rsquo;s API and an end-user&rsquo;s toolkit for handling BAM files</div></div><a href=https://github.com/pezmaster31/bamtools class=is-pretty-link>https://github.com/pezmaster31/bamtools</a></td><td></td></tr><tr><td>beagle</td><td>5.4-20241029</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for beagle</div><div class=sc-collapsible-content>Description: Beagle is a software package for phasing genotypes and for imputing ungenotyped markers</div></div><a href=https://faculty.washington.edu/browning/beagle/beagle.html class=is-pretty-link>https://faculty.washington.edu/browning/beagle/beagle.html</a></td><td></td></tr><tr><td>bedtools</td><td>2.31.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for bedtools</div><div class=sc-collapsible-content>Description: a swiss-army knife of tools for a wide-range of genomics analysis tasks</div></div><a href=https://github.com/arq5x/bedtools2 class=is-pretty-link>https://github.com/arq5x/bedtools2</a></td><td>( arch/avx512 )</td></tr><tr><td>binutils</td><td>2.42</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for binutils</div><div class=sc-collapsible-content></div></div></td><td>( arch/avx512 )</td></tr><tr><td>birch</td><td>3.90</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for birch</div><div class=sc-collapsible-content>BIRCH is a powerful framework for organizing and using bioinformatics. ; BIRCH comes with hundreds of commonly-used tools, and allows you to customize your BIRCH site by adding new ones</div></div><a href=https://home.cc.umanitoba.ca/~psgendb/index.html class=is-pretty-link>https://home.cc.umanitoba.ca/~psgendb/index.html</a></td><td></td></tr><tr><td>blastplus</td><td>2.16.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for blastplus</div><div class=sc-collapsible-content>Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences</div></div><a href=https://blast.ncbi.nlm.nih.gov/ class=is-pretty-link>https://blast.ncbi.nlm.nih.gov/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>blis</td><td>0.9.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for blis</div><div class=sc-collapsible-content>Description: BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries</div></div><a href=https://github.com/flame/blis class=is-pretty-link>https://github.com/flame/blis</a></td><td>( arch/avx512 gcc/13.2.0 ), ( arch/avx2 gcc/13.2.0 )</td></tr><tr><td>boost</td><td>1.78.0, 1.85.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for boost</div><div class=sc-collapsible-content>Description: Boost provides free peer-reviewed portable C++ source libraries, emphasizing libraries that work well with the C++ Standard Library</div></div><a href=http://www.boost.org class=is-pretty-link>http://www.boost.org</a></td><td>( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/9.5.0 ), ( arch/avx512 intel/2023.2 )</td></tr><tr><td>bowtie2</td><td>2.5.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for bowtie2</div><div class=sc-collapsible-content>Description: Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences</div></div><a href=http://bowtie-bio.sourceforge.net/bowtie2/ class=is-pretty-link>http://bowtie-bio.sourceforge.net/bowtie2/</a></td><td>( arch/avx512 )</td></tr><tr><td>buildah</td><td>1.39.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for buildah</div><div class=sc-collapsible-content>Buildah is a tool that facilitates building Open Container Initiative (OCI) container images</div></div><a href=https://buildah.io/ class=is-pretty-link>https://buildah.io/</a></td><td></td></tr><tr><td>busco</td><td>5.8.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for busco</div><div class=sc-collapsible-content>Description: BUSCO provides Assessment of assembly completeness using Universal Single Copy Orthologs</div></div><a href=https://busco.ezlab.org class=is-pretty-link>https://busco.ezlab.org</a></td><td></td></tr><tr><td>bwa</td><td>0.7.18</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for bwa</div><div class=sc-collapsible-content>Description: BWA - Burrows-Wheeler Aligner is an efficient program that aligns relatively short nucleotide sequences</div></div><a href=http://bio-bwa.sourceforge.net/ class=is-pretty-link>http://bio-bwa.sourceforge.net/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>cdo</td><td>2.5.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cdo</div><div class=sc-collapsible-content>Description: CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data</div></div><a href=https://code.mpimet.mpg.de/projects/cdo class=is-pretty-link>https://code.mpimet.mpg.de/projects/cdo</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>cfitsio</td><td>4.5.0, 4.4.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cfitsio</div><div class=sc-collapsible-content>Description: CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format</div></div><a href=http://heasarc.gsfc.nasa.gov/fitsio/ class=is-pretty-link>http://heasarc.gsfc.nasa.gov/fitsio/</a></td><td></td></tr><tr><td>cgal</td><td>6.0.1, 5.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cgal</div><div class=sc-collapsible-content>Description: The Computational Geometry Algorithms Library (CGAL) is a C++ library ; that aims to provide easy access to efficient and reliable algorithms in ; computational geometry</div></div><a href=https://www.cgal.org/ class=is-pretty-link>https://www.cgal.org/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>clhep</td><td>2.4.7.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for clhep</div><div class=sc-collapsible-content>Description: The CLHEP project is intended to be a set of HEP-specific foundation and utility classes such as random generators, physics vectors, geometry and linear algebra. CLHEP is structured in a set of packages independent of any external package</div></div><a href=https://proj-clhep.web.cern.ch/proj-clhep/ class=is-pretty-link>https://proj-clhep.web.cern.ch/proj-clhep/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>cmake</td><td>3.28.4, 3.31.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cmake</div><div class=sc-collapsible-content>Description: A cross-platform, open-source build system</div></div><a href=https://www.cmake.org class=is-pretty-link>https://www.cmake.org</a></td><td></td></tr><tr><td>code-server</td><td>4.99.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for code-server</div><div class=sc-collapsible-content>Run VS Code on any machine anywhere and access it in the browser</div></div></td><td></td></tr><tr><td>compleasm</td><td>0.2.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for compleasm</div><div class=sc-collapsible-content>Description: Compleasm: a faster and more accurate reimplementation of BUSCO, a genomics tool</div></div><a href=https://github.com/huangnengCSU/compleasm class=is-pretty-link>https://github.com/huangnengCSU/compleasm</a></td><td></td></tr><tr><td>cp2k</td><td>2024.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cp2k</div><div class=sc-collapsible-content>Description: CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and classical pair and many-body potentials</div></div><a href=https://www.cp2k.org/ class=is-pretty-link>https://www.cp2k.org/</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>cppzmq</td><td>4.10.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cppzmq</div><div class=sc-collapsible-content>Description: cppzmq is a C++ binding for libzmq</div></div><a href=https://github.com/zeromq/cppzmq class=is-pretty-link>https://github.com/zeromq/cppzmq</a></td><td></td></tr><tr><td>crest</td><td>2.12</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for crest</div><div class=sc-collapsible-content>Description: CREST is an utility/driver program for the xtb program. Originally it was designed as conformer sampling program, hence the abbreviation Conformer-Rotamer Ensemble Sampling Tool, but now offers also some utility functions for calculations with the GFNn-xTB methods. Generally the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb program) and tool for the creation and analysation of structure ensembles</div></div><a href=https://xtb-docs.readthedocs.io/en/latest/crest.html class=is-pretty-link>https://xtb-docs.readthedocs.io/en/latest/crest.html</a></td><td>( arch/avx512 intel/2023.2 )</td></tr><tr><td>ctffind</td><td>4.1.14</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ctffind</div><div class=sc-collapsible-content>CTFFIND4 is a fast and accurate defocus estimation from electron micrographs</div></div><a href=https://grigoriefflab.umassmed.edu/ctffind4 class=is-pretty-link>https://grigoriefflab.umassmed.edu/ctffind4</a></td><td>( arch/avx512 intel-one/2024.1 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 )</td></tr><tr><td>cuda</td><td>12.2.2, 12.4.1, 11.8.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cuda</div><div class=sc-collapsible-content>Description: CUDA is a parallel computing platform and programming model invented by NVIDIA</div></div><a href=https://developer.nvidia.com/cuda-zone class=is-pretty-link>https://developer.nvidia.com/cuda-zone</a></td><td>( arch/avx512 ), , ( arch/avx2 )</td></tr><tr><td>cudnn</td><td>8.8.1.3+cuda-11.8.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for cudnn</div><div class=sc-collapsible-content>Description: NVIDIA cuDNN is a GPU-accelerated library of primitives for deep neural networks</div></div><a href=https://developer.nvidia.com/cudnn class=is-pretty-link>https://developer.nvidia.com/cudnn</a></td><td></td></tr><tr><td>deepvariant</td><td>1.8.0-gpu, 1.8.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for deepvariant</div><div class=sc-collapsible-content>DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file</div></div></td><td></td></tr><tr><td>dejagnu</td><td>1.6.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for dejagnu</div><div class=sc-collapsible-content>Description: DejaGnu is a framework for testing other programs</div></div><a href=https://www.gnu.org/software/dejagnu/ class=is-pretty-link>https://www.gnu.org/software/dejagnu/</a></td><td></td></tr><tr><td>diamond</td><td>2.1.10</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for diamond</div><div class=sc-collapsible-content>Description: Accelerated BLAST compatible local sequence aligner</div></div><a href=https://github.com/bbuchfink/diamond class=is-pretty-link>https://github.com/bbuchfink/diamond</a></td><td></td></tr><tr><td>dlb</td><td>3.5.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for dlb</div><div class=sc-collapsible-content>Description: DLB is a dynamic library designed to speed up HPC hybrid applications by improving the load balance of the outer level of parallelism by dynamically redistributing the computational resources at the inner level of parallelism at run time</div></div><a href=https://pm.bsc.es/dlb/ class=is-pretty-link>https://pm.bsc.es/dlb/</a></td><td>( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>eccodes</td><td>2.31.0, 2.40.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for eccodes</div><div class=sc-collapsible-content>Description: ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding)</div></div><a href=https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home class=is-pretty-link>https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home</a></td><td>( arch/avx512 gcc/11.5.0 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>eigen</td><td>3.4.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for eigen</div><div class=sc-collapsible-content>; Description: Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms</div></div><a href="https://eigen.tuxfamily.org/index.php?title=Main_Page" class=is-pretty-link>https://eigen.tuxfamily.org/index.php?title=Main_Page</a></td><td>( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), , ( arch/avx512 )</td></tr><tr><td>esmf</td><td>8.7.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for esmf</div><div class=sc-collapsible-content>Description: The Earth System Modeling Framework (ESMF) is software for building and coupling weather, climate, and related models</div></div><a href=http://earthsystemmodeling.org class=is-pretty-link>http://earthsystemmodeling.org</a></td><td>( arch/avx512 intel/2023.2 openmpi/4.1.6 )</td></tr><tr><td>espresso</td><td>7.3.1, 7.3.1+aocl-4.2.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for espresso</div><div class=sc-collapsible-content>Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft ; Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft)</div></div><a href=https://www.quantum-espresso.org, class=is-pretty-link>https://www.quantum-espresso.org,</a></td><td>( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>expect</td><td>5.45.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for expect</div><div class=sc-collapsible-content>Description: Expect is a tool for automating interactive applications such as telnet, ftp, passwd, fsck, rlogin, tip, etc</div></div><a href=http://expect.sourceforge.net/ class=is-pretty-link>http://expect.sourceforge.net/</a></td><td></td></tr><tr><td>fastqc</td><td>0.12.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for fastqc</div><div class=sc-collapsible-content>Description: FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline</div></div><a href=https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ class=is-pretty-link>https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></td><td></td></tr><tr><td>feko</td><td>2021.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for feko</div><div class=sc-collapsible-content>Altair Feko high-frequency electromagnetic simulation software for Okhmatovsky group</div></div></td><td></td></tr><tr><td>ffmpeg</td><td>7.0.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ffmpeg</div><div class=sc-collapsible-content>A statically build binary for ffmpeg Linux</div></div><a href=https://johnvansickle.com/ffmpeg/ class=is-pretty-link>https://johnvansickle.com/ffmpeg/</a></td><td></td></tr><tr><td>fftw</td><td>3.3.10</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for fftw</div><div class=sc-collapsible-content>Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)</div></div><a href=http://www.fftw.org class=is-pretty-link>http://www.fftw.org</a></td><td>( arch/avx512 gcc/11.5.0 ), ( arch/avx512 gcc/13.2.0 openmpi/5.0.6 ), ( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 aocc/4.2.0 openmpi/4.1.6 ), , ( cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 aocc/4.2.0 ), ( arch/avx512 gcc/9.5.0 openmpi/4.1.6 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 intel/2023.2 ), ( arch/avx512 intel-one/2024.1 openmpi/4.1.6 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel-one/2024.1 openmpi/5.0.6 )</td></tr><tr><td>flex</td><td>2.6.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for flex</div><div class=sc-collapsible-content>Description: Flex is a tool for generating scanners</div></div><a href=https://github.com/westes/flex class=is-pretty-link>https://github.com/westes/flex</a></td><td></td></tr><tr><td>flexpart</td><td>11</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for flexpart</div><div class=sc-collapsible-content>Description: FLEXPART (FLEXible PARTicle dispersion model) is a Lagrangian transport and dispersion model suitable for the simulation of a large range of atmospheric transport processes</div></div><a href=https://www.flexpart.eu class=is-pretty-link>https://www.flexpart.eu</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>freeglut</td><td>3.4.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for freeglut</div><div class=sc-collapsible-content>Description: freeglut is a free-software/open-source alternative to the OpenGL Utility Toolkit (GLUT) library</div></div><a href=https://freeglut.sourceforge.net/ class=is-pretty-link>https://freeglut.sourceforge.net/</a></td><td></td></tr><tr><td>gate</td><td>9.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gate</div><div class=sc-collapsible-content>Description: GATE is an advanced opensource software developed by the international OpenGATE collaboration and dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography</div></div><a href=http://www.opengatecollaboration.org/ class=is-pretty-link>http://www.opengatecollaboration.org/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>gatk</td><td>4.6.1.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gatk</div><div class=sc-collapsible-content>Description: The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size</div></div><a href=http://www.broadinstitute.org/gatk/ class=is-pretty-link>http://www.broadinstitute.org/gatk/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>gaussian</td><td>g16.c01, g16.b01</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gaussian</div><div class=sc-collapsible-content>Gaussian 16.c01 - A quantum chemistry package</div></div></td><td></td></tr><tr><td>gcc</td><td>13.2.0, 8.5.0, 12.3.0, 11.5.0, 9.5.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gcc</div><div class=sc-collapsible-content>Description: The GNU Compiler Collection for C, C++, and Fortran</div></div><a href=https://gcc.gnu.org class=is-pretty-link>https://gcc.gnu.org</a></td><td>( arch/avx2 ), , ( cuda/12.4.1 arch/avx2 ), ( cuda/12.2.2 arch/avx2 ), ( arch/avx512 ), ( cuda/11.8.0 arch/avx2 )</td></tr><tr><td>gdal</td><td>3.10.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gdal</div><div class=sc-collapsible-content>Description: GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for data translation and processing</div></div><a href=https://www.gdal.org/ class=is-pretty-link>https://www.gdal.org/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>geant4</td><td>11.2.2, 11.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for geant4</div><div class=sc-collapsible-content>Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science</div></div><a href=http://geant4.cern.ch/ class=is-pretty-link>http://geant4.cern.ch/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>geos</td><td>3.13.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for geos</div><div class=sc-collapsible-content>Description: GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS)</div></div><a href=https://trac.osgeo.org/geos class=is-pretty-link>https://trac.osgeo.org/geos</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>git</td><td>2.49.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for git</div><div class=sc-collapsible-content>Description: Git is a free and open source distributed version control system</div></div><a href=http://git-scm.com class=is-pretty-link>http://git-scm.com</a></td><td></td></tr><tr><td>git-annex</td><td>10.20250320</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for git-annex</div><div class=sc-collapsible-content>Git-annex allows managing large files with git, without storing the file contents in git. ; It can sync, backup, and archive your data, offline and online. ; Checksums and encryption keep your data safe and secure</div></div><a href=https://git-annex.branchable.com/ class=is-pretty-link>https://git-annex.branchable.com/</a></td><td></td></tr><tr><td>git-lfs</td><td>3.6.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for git-lfs</div><div class=sc-collapsible-content>Git Large File Storage ; Git LFS is a command line extension and specification for managing large files with Git</div></div><a href=https://git-lfs.com/ class=is-pretty-link>https://git-lfs.com/</a></td><td></td></tr><tr><td>globus</td><td>3.33.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for globus</div><div class=sc-collapsible-content>Tools to interact with Globus using the command line interface: ; - Globus CLI (<a href=https://github.com/globus/globus-cli class=is-pretty-link>https://github.com/globus/globus-cli</a>
) ; - Globus Connect Personal (<a href=https://www.globus.org/globus-connect-personal class=is-pretty-link>https://www.globus.org/globus-connect-personal</a>
)</div></div><a href=https://www.globus.org/ class=is-pretty-link>https://www.globus.org/</a></td><td></td></tr><tr><td>glost</td><td>0.3.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for glost</div><div class=sc-collapsible-content>Description: This is GLOST, the Greedy Launcher Of Small Tasks</div></div><a href=https://github.com/cea-hpc/glost class=is-pretty-link>https://github.com/cea-hpc/glost</a></td><td>( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>glpk</td><td>5.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for glpk</div><div class=sc-collapsible-content>Description: The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems. It is a set of routines written in ANSI C and organized in the form of a callable library</div></div><a href=https://www.gnu.org/software/glpk/ class=is-pretty-link>https://www.gnu.org/software/glpk/</a></td><td>( arch/avx512 intel/2023.2 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/9.5.0 )</td></tr><tr><td>gmp</td><td>6.3.0, 6.2.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gmp</div><div class=sc-collapsible-content>Description: GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating point numbers</div></div><a href=https://gmplib.org/ class=is-pretty-link>https://gmplib.org/</a></td><td>( arch/avx512 gcc/13.2.0 ), , ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 )</td></tr><tr><td>gnina</td><td>2024, 1.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gnina</div><div class=sc-collapsible-content>gnina (pronounced NEE-na) is a molecular docking program with integrated support for scoring and optimizing ligands using convolutional neural networks</div></div><a href=https://github.com/gnina/gnina class=is-pretty-link>https://github.com/gnina/gnina</a></td><td></td></tr><tr><td>gnuplot</td><td>5.4.2, 6.0.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gnuplot</div><div class=sc-collapsible-content>Description: Gnuplot is a portable command-line driven graphing utility for Linux, OS/2, MS Windows, OSX, VMS, and many other platforms</div></div><a href=http://www.gnuplot.info class=is-pretty-link>http://www.gnuplot.info</a></td><td></td></tr><tr><td>golang</td><td>1.24.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for golang</div><div class=sc-collapsible-content>Go is an opensource programming language supported by Google</div></div></td><td></td></tr><tr><td>grace</td><td>5.99.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for grace</div><div class=sc-collapsible-content>Description: Grace is a WYSIWYG 2D plotting tool for X Windows System and Motif</div></div><a href=http://freecode.com/projects/grace class=is-pretty-link>http://freecode.com/projects/grace</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>gromacs</td><td>2021.6, 2022, 2024.1, 2023.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gromacs</div><div class=sc-collapsible-content>Description: GROMACS (GROningen MAchine for Chemical Simulations) is a molecular dynamics package primarily designed for simulations of proteins, lipids and nucleic acids</div></div><a href=http://www.gromacs.org class=is-pretty-link>http://www.gromacs.org</a></td><td>( arch/avx512 gcc/11.5.0 openmpi/4.1.6 ), ( arch/avx512 aocc/4.2.0 openmpi/4.1.6 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 )</td></tr><tr><td>gsl</td><td>2.7</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for gsl</div><div class=sc-collapsible-content>Description: The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers</div></div><a href=http://www.gnu.org/software/gsl class=is-pretty-link>http://www.gnu.org/software/gsl</a></td><td>( arch/avx512 intel/2023.2 ), ( arch/avx2 gcc/13.2.0 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 gcc/9.5.0 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>hdf5</td><td>1.12.3, 1.10.9, 1.14.2, 1.14.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for hdf5</div><div class=sc-collapsible-content>Description: HDF5 is a data model, library, and file format for storing and managing data</div></div><a href=https://portal.hdfgroup.org class=is-pretty-link>https://portal.hdfgroup.org</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/9.5.0 ), ( cuda/12.4.1 arch/avx2 gcc/12.3.0 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 gcc/11.5.0 ), ( arch/avx512 gcc/13.2.0 openmpi/5.0.6 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel-one/2024.1 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6 ), ( arch/avx512 intel/2023.2 intelmpi/2021.10 ), ( arch/avx512 intel-one/2024.1 openmpi/5.0.6 )</td></tr><tr><td>highfive</td><td>2.10.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for highfive</div><div class=sc-collapsible-content>Description: HighFive is a modern header-only C++11 friendly interface for libhdf5</div></div><a href=https://github.com/BlueBrain/HighFive class=is-pretty-link>https://github.com/BlueBrain/HighFive</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>hisat2</td><td>2.2.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for hisat2</div><div class=sc-collapsible-content>Description: HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) against the general human population (as well as against a single reference genome)</div></div><a href=https://daehwankimlab.github.io/hisat2/ class=is-pretty-link>https://daehwankimlab.github.io/hisat2/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>hmmer</td><td>3.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for hmmer</div><div class=sc-collapsible-content>Description: HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs)</div></div><a href=http://hmmer.org/ class=is-pretty-link>http://hmmer.org/</a></td><td></td></tr><tr><td>htgettoken</td><td>2.0-2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for htgettoken</div><div class=sc-collapsible-content>htgettoken gets OIDC bearer tokens by interacting with Hashicorp vault</div></div><a href=https://github.com/fermitools/htgettoken class=is-pretty-link>https://github.com/fermitools/htgettoken</a></td><td></td></tr><tr><td>imb</td><td>2021.7</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for imb</div><div class=sc-collapsible-content>; IMB is Intel suite for MPI benchmarks. ; This module only includes classic C benchmarks that can be built by most of compilers. ; ; ; IMB is Intel suite for MPI benchmarks. ; This module only includes classic C benchmarks that can be built by most of compilers</div></div><a href=https://github.com/intel/mpi-benchmarks, class=is-pretty-link>https://github.com/intel/mpi-benchmarks,</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 intelmpi/2021.10 ), ( arch/avx2 intel/2023.2 intelmpi/2021.10 )</td></tr><tr><td>intel</td><td>2019.5, 2023.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for intel</div><div class=sc-collapsible-content>; Description: Intel compilers for C, C++, and Fortran ; ; Description: Intel compilers for C, C++, and Fortran</div></div><a href=https://software.intel.com/en-us/parallel-studio-xe class=is-pretty-link>https://software.intel.com/en-us/parallel-studio-xe</a></td><td>( arch/avx512 ), ( arch/avx2 ),</td></tr><tr><td>intel-one</td><td>2024.1, 2023.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for intel-one</div><div class=sc-collapsible-content>; Description: Intel compilers for C, C++, and Fortran from New OneAPI suite ; ; ; Description: Intel compilers for C, C++, and Fortran from New OneAPI suite</div></div><a href=https://software.intel.com/content/www/us/en/develop/tools/oneapi.html class=is-pretty-link>https://software.intel.com/content/www/us/en/develop/tools/oneapi.html</a></td><td>, ( arch/avx512 )</td></tr><tr><td>intelmpi</td><td>2021.10, 2019.8</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for intelmpi</div><div class=sc-collapsible-content>Description: Intel MPI library with compiler wrappers for C, C++, and Fortran</div></div><a href=https://software.intel.com/en-us/intel-mpi-library class=is-pretty-link>https://software.intel.com/en-us/intel-mpi-library</a></td><td>, ( arch/avx2 intel/2023.2 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 intel/2023.2 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/11.5.0 ), ( arch/avx2 gcc/13.2.0 )</td></tr><tr><td>jags</td><td>4.3.2+mkl-2019.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for jags</div><div class=sc-collapsible-content>Description: JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation</div></div><a href=http://mcmc-jags.sourceforge.net/ class=is-pretty-link>http://mcmc-jags.sourceforge.net/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>jasper</td><td>4.0.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for jasper</div><div class=sc-collapsible-content>Description: The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard</div></div><a href=https://www.ece.uvic.ca/~frodo/jasper/ class=is-pretty-link>https://www.ece.uvic.ca/~frodo/jasper/</a></td><td>( arch/avx512 gcc/11.5.0 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>jellyfish</td><td>2.3.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for jellyfish</div><div class=sc-collapsible-content>Description: Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA</div></div><a href=http://www.genome.umd.edu/jellyfish.html class=is-pretty-link>http://www.genome.umd.edu/jellyfish.html</a></td><td></td></tr><tr><td>jemalloc</td><td>5.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for jemalloc</div><div class=sc-collapsible-content>Description: jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and scalable concurrency support</div></div><a href=http://jemalloc.net class=is-pretty-link>http://jemalloc.net</a></td><td></td></tr><tr><td>jq</td><td>1.7</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for jq</div><div class=sc-collapsible-content>Description: jq is a lightweight and flexible command-line JSON processor</div></div><a href=https://stedolan.github.io/jq/ class=is-pretty-link>https://stedolan.github.io/jq/</a></td><td></td></tr><tr><td>julia</td><td>1.11.3, 1.10.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for julia</div><div class=sc-collapsible-content>Description: The Julia Language: A fresh approach to technical computing</div></div><a href=http://julialang.org class=is-pretty-link>http://julialang.org</a></td><td></td></tr><tr><td>kim</td><td>2.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for kim</div><div class=sc-collapsible-content>Open Knowledgebase of Interatomic Models (KIM)</div></div></td><td>( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx2 gcc/13.2.0 )</td></tr><tr><td>kokkos</td><td>4.4.01</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for kokkos</div><div class=sc-collapsible-content>KOKKOS is a portable, hardware agnostic layer for writing C++ HPC applications</div></div></td><td>( cuda/12.4.1 arch/avx2 gcc/13.2.0 )</td></tr><tr><td>lammps</td><td>2024-08-29p1-nep, 2024-08-29p1, 2021-09-29</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for lammps</div><div class=sc-collapsible-content>Description: LAMMPS stands for Large-scale Atomic/Molecular Massively Parallel Simulator</div></div><a href=http://www.lammps.org/ class=is-pretty-link>http://www.lammps.org/</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel-one/2024.1 openmpi/4.1.6 )</td></tr><tr><td>libaec</td><td>1.0.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for libaec</div><div class=sc-collapsible-content>Description: Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa</div></div><a href=https://gitlab.dkrz.de/k202009/libaec class=is-pretty-link>https://gitlab.dkrz.de/k202009/libaec</a></td><td></td></tr><tr><td>libint-cp2k</td><td>2.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for libint-cp2k</div><div class=sc-collapsible-content>Description: Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory</div></div><a href=https://github.com/evaleev/libint class=is-pretty-link>https://github.com/evaleev/libint</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>libvori</td><td>220621</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for libvori</div><div class=sc-collapsible-content>libvori is a Library for Voronoi Integration written in C++</div></div></td><td></td></tr><tr><td>libxc</td><td>6.2.2, 5.1.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for libxc</div><div class=sc-collapsible-content>Description: Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals</div></div><a href=https://www.tddft.org/programs/octopus/wiki/index.php/Libxc class=is-pretty-link>https://www.tddft.org/programs/octopus/wiki/index.php/Libxc</a></td><td>( arch/avx512 intel-one/2024.1 ), ( arch/avx512 intel/2023.2 ), ( arch/avx512 aocc/4.2.0 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>libxml2</td><td>2.11.9</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for libxml2</div><div class=sc-collapsible-content>Description: libxml2 is an XML toolkit implemented in C, originally developed for the GNOME Project</div></div><a href=https://gitlab.gnome.org/GNOME/libxml2 class=is-pretty-link>https://gitlab.gnome.org/GNOME/libxml2</a></td><td></td></tr><tr><td>matlab</td><td>R2024A, R2020B2, R2022A, R2023B</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for matlab</div><div class=sc-collapsible-content>MATLAB, a computing environment from Mathworks</div></div></td><td></td></tr><tr><td>mcr</td><td>R2024a, R2022a, R2020b, R2023b</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mcr</div><div class=sc-collapsible-content>Description: Run MATLAB applications on computers that do not have MATLAB installed</div></div><a href=https://www.mathworks.com/help/compiler/matlab-runtime.html class=is-pretty-link>https://www.mathworks.com/help/compiler/matlab-runtime.html</a></td><td></td></tr><tr><td>megahit</td><td>1.2.9</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for megahit</div><div class=sc-collapsible-content>Description: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph</div></div><a href=https://github.com/voutcn/megahit class=is-pretty-link>https://github.com/voutcn/megahit</a></td><td></td></tr><tr><td>metashape-pro</td><td>2.1.3, 2.2.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for metashape-pro</div><div class=sc-collapsible-content>Agisoft Metashape is a stand-alone software product that performs photogrammetric processing of digital images ; and generates 3D spatial data to be used in GIS applications, cultural heritage documentation, ; and visual effects production as well as for indirect measurements of objects of various scales</div></div><a href=https://www.agisoft.com/ class=is-pretty-link>https://www.agisoft.com/</a></td><td></td></tr><tr><td>metis</td><td>5.1.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for metis</div><div class=sc-collapsible-content>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices</div></div><a href=http://glaros.dtc.umn.edu/gkhome/metis/metis/overview class=is-pretty-link>http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</a></td><td>( arch/avx512 intel/2023.2 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/9.5.0 )</td></tr><tr><td>metis32</td><td>5.1.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for metis32</div><div class=sc-collapsible-content>Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices</div></div><a href=http://glaros.dtc.umn.edu/gkhome/metis/metis/overview class=is-pretty-link>http://glaros.dtc.umn.edu/gkhome/metis/metis/overview</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>micro</td><td>2.0.14</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for micro</div><div class=sc-collapsible-content>micro is a simple, modern and intuitive text editor</div></div></td><td></td></tr><tr><td>mii</td><td>1.1.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mii</div><div class=sc-collapsible-content>Description: A smart search engine for module environments</div></div><a href=https://github.com/codeandkey/mii class=is-pretty-link>https://github.com/codeandkey/mii</a></td><td></td></tr><tr><td>minimap2</td><td>2.28</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for minimap2</div><div class=sc-collapsible-content>Minimap2 is a versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database</div></div></td><td></td></tr><tr><td>mkl</td><td>2019.5, 2024.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mkl</div><div class=sc-collapsible-content>Description: Intel Math Kernel Library</div></div><a href=https://software.intel.com/en-us/mkl class=is-pretty-link>https://software.intel.com/en-us/mkl</a></td><td>( arch/avx512 gcc/9.5.0 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( cuda/11.8.0 arch/avx2 gcc/11.5.0 ), ( arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/11.5.0 )</td></tr><tr><td>molden</td><td>7.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for molden</div><div class=sc-collapsible-content>Description: Molden is a package for displaying Molecular Density from the Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical packages Mopac/Ampac</div></div><a href=http://www.cmbi.ru.nl/molden/ class=is-pretty-link>http://www.cmbi.ru.nl/molden/</a></td><td></td></tr><tr><td>mpfr</td><td>4.2.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mpfr</div><div class=sc-collapsible-content>Description: The MPFR library is a C library for multiple-precision ; floating-point computations with correct rounding</div></div><a href=https://www.mpfr.org/ class=is-pretty-link>https://www.mpfr.org/</a></td><td>( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 ),</td></tr><tr><td>mumax3</td><td>3.10</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mumax3</div><div class=sc-collapsible-content>Mumax3 GPU-accelerated micromagnetic simulation program for CUDA</div></div></td><td></td></tr><tr><td>mummer</td><td>4.0.0rc1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mummer</div><div class=sc-collapsible-content>Description: MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. AMOS makes use of it</div></div><a href=http://mummer.sourceforge.net/ class=is-pretty-link>http://mummer.sourceforge.net/</a></td><td></td></tr><tr><td>mustang</td><td>3.2.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for mustang</div><div class=sc-collapsible-content>Description: MUSTANG (MUltiple STructural AligNment AlGorithm), for the alignment of multiple protein structures</div></div><a href=http://www.csse.monash.edu.au/~karun/Site/mustang.html class=is-pretty-link>http://www.csse.monash.edu.au/~karun/Site/mustang.html</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>nbo</td><td>nbo7-2021</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for nbo</div><div class=sc-collapsible-content>The Natural Bond Orbital (NBO) program NBO 7</div></div></td><td></td></tr><tr><td>nco</td><td>5.3.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for nco</div><div class=sc-collapsible-content>Description: The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, HDF5, and, most recently, Zarr</div></div><a href=https://nco.sourceforge.net class=is-pretty-link>https://nco.sourceforge.net</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>ncview</td><td>2.1.11</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ncview</div><div class=sc-collapsible-content>Description: Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button ; look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc</div></div><a href=http://meteora.ucsd.edu/~pierce/ncview_home_page.html class=is-pretty-link>http://meteora.ucsd.edu/~pierce/ncview_home_page.html</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>netcdf</td><td>4.9.2+hdf5-1.14.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for netcdf</div><div class=sc-collapsible-content>Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data</div></div><a href=http://www.unidata.ucar.edu/software/netcdf class=is-pretty-link>http://www.unidata.ucar.edu/software/netcdf</a></td><td>( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/11.5.0 ), ( arch/avx512 intel/2023.2 )</td></tr><tr><td>nextflow</td><td>24.10.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for nextflow</div><div class=sc-collapsible-content>Nextflow is a reactive workflow framework and a programming DSL that eases writing computational pipelines with complex data</div></div><a href=https://www.nextflow.io/ class=is-pretty-link>https://www.nextflow.io/</a></td><td></td></tr><tr><td>ninja</td><td>1.10.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ninja</div><div class=sc-collapsible-content>Description: Ninja is a small build system with a focus on speed</div></div><a href=https://ninja-build.org class=is-pretty-link>https://ninja-build.org</a></td><td></td></tr><tr><td>nodejs</td><td>18.20.2, 22.5.1, 22.11.0, 20.12.2, 20.16.0, 18.20.4, 20.18.1, 18.20.5, 20.15.1, 22.4.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for nodejs</div><div class=sc-collapsible-content>Node.JS - the asynchronous server-side JavaScript language. <a href=https://nodejs.org class=is-pretty-link>https://nodejs.org</a></div></div></td><td></td></tr><tr><td>nvtop</td><td>3.1.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for nvtop</div><div class=sc-collapsible-content>Description: NVTOP stands for Neat Videocard TOP, a (h)top like task monitor for GPUs and accelerators. It can handle multiple GPUs and print information about them in a htop-familiar way</div></div><a href=https://github.com/Syllo/nvtop class=is-pretty-link>https://github.com/Syllo/nvtop</a></td><td></td></tr><tr><td>nwchem</td><td>7.2.2+aocl-4.2.0-64, 7.2.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for nwchem</div><div class=sc-collapsible-content>Description: NWChem is an open source high performance computing package for computational chemistry</div></div></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 aocc/4.2.0 openmpi/4.1.6 ), ( arch/avx512 intel-one/2024.1 openmpi/4.1.6 )</td></tr><tr><td>openbabel</td><td>3.1.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for openbabel</div><div class=sc-collapsible-content>Description: Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It&rsquo;s an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas</div></div><a href=https://openbabel.org/ class=is-pretty-link>https://openbabel.org/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>openblas</td><td>0.3.26, 0.3.28</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for openblas</div><div class=sc-collapsible-content>Description: OpenBLAS: An optimized BLAS library</div></div><a href=https://www.openblas.net class=is-pretty-link>https://www.openblas.net</a></td><td>( cuda/12.4.1 arch/avx2 gcc/12.3.0 ), ( cuda/11.8.0 arch/avx2 gcc/11.5.0 ), ( arch/avx512 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 aocc/4.2.0 ), ( arch/avx512 intel/2023.2 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>openeye</td><td>2022.2.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for openeye</div><div class=sc-collapsible-content>OpenEye: module for R. Davis Group only</div></div></td><td></td></tr><tr><td>openjdk</td><td>17.0.12_7, 17.0.13_11, 21.0.2, 21.0.5_11, 21.0.6_7, 21.0.3_9, 21.0.4_7, 17.0.11_9, 11.0.22</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for openjdk</div><div class=sc-collapsible-content>Description: The free and opensource java implementation</div></div><a href=https://jdk.java.net class=is-pretty-link>https://jdk.java.net</a></td><td></td></tr><tr><td>openmpi</td><td>4.1.6, 5.0.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for openmpi</div><div class=sc-collapsible-content>Description: An open source Message Passing Interface implementation</div></div><a href=http://www.open-mpi.org class=is-pretty-link>http://www.open-mpi.org</a></td><td>( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx512 intel-one/2024.1 ), ( arch/avx512 intel/2023.2 ), ( arch/avx2 intel/2023.2 ), ( arch/avx512 gcc/11.5.0 ), ( arch/avx512 aocc/4.2.0 ), ( arch/avx512 gcc/13.2.0 ), ( cuda/12.2.2 arch/avx2 gcc/12.3.0 ), ( arch/avx2 aocc/4.2.0 ), ( cuda/12.4.1 arch/avx2 gcc/12.3.0 ), ( arch/avx512 gcc/9.5.0 )</td></tr><tr><td>opennurbs</td><td>8.12</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for opennurbs</div><div class=sc-collapsible-content>Description: The openNURBS Initiative provides CAD, CAM, CAE, and computer graphics software developers the tools to accurately transfer 3D geometry between applications</div></div><a href=https://github.com/mcneel/opennurbs class=is-pretty-link>https://github.com/mcneel/opennurbs</a></td><td>( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/9.5.0 )</td></tr><tr><td>openssl</td><td>3.4.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for openssl</div><div class=sc-collapsible-content>Description: OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols</div></div><a href=http://www.openssl.org class=is-pretty-link>http://www.openssl.org</a></td><td></td></tr><tr><td>orca</td><td>5.0.4, 6.0.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for orca</div><div class=sc-collapsible-content>Description ; =========== ; ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum ; chemistry with specific emphasis on spectroscopic properties of open-shell molecules. ; It features a wide variety of standard quantum chemical methods ranging from ; semiempirical methods to DFT to single- and multireference correlated ab initio methods. ; It can also treat environmental and relativistic effects</div></div><a href=http://cec.mpg.de/forum/ class=is-pretty-link>http://cec.mpg.de/forum/</a></td><td>( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>osu-micro-benchmarks</td><td>5.7.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for osu-micro-benchmarks</div><div class=sc-collapsible-content>Description: The Ohio MicroBenchmark suite is a collection of independent MPI message passing performance microbenchmarks developed and written at The Ohio State University</div></div><a href=http://mvapich.cse.ohio-state.edu/benchmarks/ class=is-pretty-link>http://mvapich.cse.ohio-state.edu/benchmarks/</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>ovito</td><td>3.12.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ovito</div><div class=sc-collapsible-content>OVITO (Open Visualization Tool) is a scientific data visualization and analysis software designed for molecular and particle-based simulation models</div></div></td><td></td></tr><tr><td>pandoc</td><td>3.6.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for pandoc</div><div class=sc-collapsible-content>Description: If you need to convert files from one markup format into another, pandoc is your swiss-army knife</div></div><a href=https://pandoc.org class=is-pretty-link>https://pandoc.org</a></td><td></td></tr><tr><td>paraview-offscreen</td><td>5.10.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for paraview-offscreen</div><div class=sc-collapsible-content>Description: Off-screen rendering with ParaView, a scientific parallel visualizer</div></div><a href=http://www.paraview.org class=is-pretty-link>http://www.paraview.org</a></td><td></td></tr><tr><td>parmetis</td><td>4.0.3-shared</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for parmetis</div><div class=sc-collapsible-content>Description: ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices</div></div><a href=http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview class=is-pretty-link>http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview</a></td><td>( arch/avx512 intel/2023.2 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>perl</td><td>5.40.1, 5.38.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for perl</div><div class=sc-collapsible-content>Description: Perl 5 is a highly capable, feature-rich programming language with over 27 years of development</div></div><a href=http://www.perl.org class=is-pretty-link>http://www.perl.org</a></td><td>( arch/avx2 ), ( arch/avx512 )</td></tr><tr><td>petsc-complex</td><td>3.13.3+mkl-2019.5, 3.22.2+mkl-2019.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for petsc-complex</div><div class=sc-collapsible-content>Description: PETSc is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations</div></div><a href=http://www.mcs.anl.gov/petsc/index.html class=is-pretty-link>http://www.mcs.anl.gov/petsc/index.html</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>petsc-real</td><td>3.22.2+mkl-2019.5, 3.13.3+mkl-2019.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for petsc-real</div><div class=sc-collapsible-content>Description: PETSc is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations</div></div><a href=http://www.mcs.anl.gov/petsc/index.html class=is-pretty-link>http://www.mcs.anl.gov/petsc/index.html</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>picard</td><td>3.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for picard</div><div class=sc-collapsible-content>Picard: A set of tools (in Java) for working with next generation sequencing data in the BAM format</div></div><a href=https://broadinstitute.github.io/picard/ class=is-pretty-link>https://broadinstitute.github.io/picard/</a></td><td></td></tr><tr><td>pnetcdf</td><td>1.14.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for pnetcdf</div><div class=sc-collapsible-content>Description: Parallel netCDF: A Parallel I/O Library for NetCDF File Access</div></div><a href=https://trac.mcs.anl.gov/projects/parallel-netcdf class=is-pretty-link>https://trac.mcs.anl.gov/projects/parallel-netcdf</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 openmpi/4.1.6 )</td></tr><tr><td>podman</td><td>5.4.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for podman</div><div class=sc-collapsible-content>Podman (the POD MANager) is a tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers</div></div><a href=https://podman.io/ class=is-pretty-link>https://podman.io/</a></td><td></td></tr><tr><td>podman-compose</td><td>1.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for podman-compose</div><div class=sc-collapsible-content>An implementation of Compose Spec with Podman backend</div></div><a href=https://github.com/containers/podman-compose class=is-pretty-link>https://github.com/containers/podman-compose</a></td><td></td></tr><tr><td>podman-tui</td><td>1.5.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for podman-tui</div><div class=sc-collapsible-content>Podman-tui is a terminal user interface for podman environment. ; It uses podman go bindings to communicate with local or remote podman machine (through SSH)</div></div><a href=https://github.com/containers/podman-tui/ class=is-pretty-link>https://github.com/containers/podman-tui/</a></td><td></td></tr><tr><td>prodigal</td><td>2.6.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for prodigal</div><div class=sc-collapsible-content>Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee</div></div><a href=https://github.com/hyattpd/Prodigal/ class=is-pretty-link>https://github.com/hyattpd/Prodigal/</a></td><td></td></tr><tr><td>proj</td><td>9.5.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for proj</div><div class=sc-collapsible-content>Description: Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates</div></div><a href=https://proj.org class=is-pretty-link>https://proj.org</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>prsice</td><td>2.3.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for prsice</div><div class=sc-collapsible-content>Description: PRSice (pronounced &lsquo;precise&rsquo;) is a software package for calculating, applying, evaluating and plotting the results of polygenic risk scores (PRS). PRSice can run at high-resolution to provide the best-fit PRS as well as provide results calculated at broad P-value thresholds, illustrating results corresponding to either, can thin SNPs according to linkage disequilibrium and P-value ( clumping ), and can be applied across multiple traits in a single run</div></div><a href=https://github.com/choishingwan/PRSice/tree/master class=is-pretty-link>https://github.com/choishingwan/PRSice/tree/master</a></td><td>( arch/avx512 )</td></tr><tr><td>python</td><td>3.11.11, 3.10.14, 3.11.8, 3.10.16, 3.12.9</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for python</div><div class=sc-collapsible-content>Description: The Python programming language</div></div><a href=https://www.python.org/ class=is-pretty-link>https://www.python.org/</a></td><td>( arch/avx512 gcc/13.2.0 ), ( arch/avx512 gcc/11.5.0 ), ( arch/avx2 gcc/13.2.0 ), ( arch/avx512 intel-one/2024.1 ), ( cuda/12.4.1 arch/avx2 gcc/12.3.0 ), , ( arch/avx512 aocc/4.2.0 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( cuda/12.2.2 arch/avx2 gcc/12.3.0 ), ( arch/avx512 gcc/9.5.0 ), ( cuda/11.8.0 arch/avx2 gcc/11.5.0 )</td></tr><tr><td>qiime2</td><td>2024.10</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for qiime2</div><div class=sc-collapsible-content>Description: QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data</div></div><a href=http://qiime2.org/ class=is-pretty-link>http://qiime2.org/</a></td><td></td></tr><tr><td>qt</td><td>6.7.1, 6.8.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for qt</div><div class=sc-collapsible-content>Description: Qt is a comprehensive cross-platform C++ application framework</div></div><a href=https://qt.io/ class=is-pretty-link>https://qt.io/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>quast</td><td>5.3.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for quast</div><div class=sc-collapsible-content>Description: QUAST is an open-source bioinformatics Quality Assessment Tool for Genome Assemblies</div></div></td><td></td></tr><tr><td>r</td><td>4.4.1+mkl-2019.5, 4.4.1+aocl-4.2.0, 4.4.1+mkl-2024.1, 4.3.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for r</div><div class=sc-collapsible-content>Description: R is a language and environment for statistical computing and graphics</div></div><a href=https://www.r-project.org class=is-pretty-link>https://www.r-project.org</a></td><td>( arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 ), ( arch/avx2 ), ( arch/avx512 ),</td></tr><tr><td>ramalama</td><td>0.7.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ramalama</div><div class=sc-collapsible-content>RamaLama tool facilitates local management and serving of AI Models</div></div><a href=https://github.com/containers/ramalama class=is-pretty-link>https://github.com/containers/ramalama</a></td><td></td></tr><tr><td>ratarmount</td><td>1.0.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for ratarmount</div><div class=sc-collapsible-content>Random Access To Archived Resources (Ratarmount) ; Ratarmount collects all file positions inside a TAR so that it can easily jump to and read from any file without extracting it. ; It, then, mounts the TAR using fusepy for read access just like archivemount. ; In contrast to libarchive, on which archivemount is based, random access and true seeking is supported. ; And in contrast to tarindexer, which also collects file positions for random access, ratarmount offers easy access via FUSE and support for compressed TARs</div></div><a href=https://github.com/mxmlnkn/ratarmount class=is-pretty-link>https://github.com/mxmlnkn/ratarmount</a></td><td></td></tr><tr><td>rclone</td><td>1.67.0, 1.68.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for rclone</div><div class=sc-collapsible-content>Rclone: sync files/directories to and from a variety of online storage services</div></div></td><td></td></tr><tr><td>relion</td><td>4.0.2, 5.0.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for relion</div><div class=sc-collapsible-content>Description: RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer program for Maximum. A Posteriori refinement of (multiple) 3D reconstructions or 2D class averages in cryo-electron microscopy</div></div></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel-one/2024.1 openmpi/5.0.6 ), ( cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6 )</td></tr><tr><td>root</td><td>6.34.02, 6.32.08</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for root</div><div class=sc-collapsible-content>Description: The ROOT system provides a set of OO frameworks with all the functionality needed to handle and analyze large amounts of data in a very efficient way</div></div><a href=https://root.cern.ch/drupal/ class=is-pretty-link>https://root.cern.ch/drupal/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>rs-server</td><td>2024.12.1-563</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for rs-server</div><div class=sc-collapsible-content>RStudio server for use with Ondemand or Jupyter</div></div></td><td></td></tr><tr><td>rust</td><td>1.86.0, 1.79.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for rust</div><div class=sc-collapsible-content>Description: Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety</div></div><a href=https://www.rust-lang.org class=is-pretty-link>https://www.rust-lang.org</a></td><td></td></tr><tr><td>samtools</td><td>1.20</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for samtools</div><div class=sc-collapsible-content>Description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format</div></div><a href=http://www.htslib.org class=is-pretty-link>http://www.htslib.org</a></td><td>( arch/avx512 gcc/13.2.0 ),</td></tr><tr><td>scipy-bundle</td><td>2023+python-3.10.14, 2023+python-3.10.16, 2023+python-3.11.8</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for scipy-bundle</div><div class=sc-collapsible-content>A bundle of common Python packages: Numpy, Scipy, Pandas, Matplotlib etc</div></div></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>scons</td><td>3.1.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for scons</div><div class=sc-collapsible-content>Description: SCons is a software construction tool</div></div><a href=http://scons.org class=is-pretty-link>http://scons.org</a></td><td></td></tr><tr><td>scotch</td><td>7.0.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for scotch</div><div class=sc-collapsible-content>Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning</div></div><a href=https://www.labri.fr/perso/pelegrin/scotch/ class=is-pretty-link>https://www.labri.fr/perso/pelegrin/scotch/</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 openmpi/4.1.6 )</td></tr><tr><td>singularity</td><td>4.1.2, 4.2.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for singularity</div><div class=sc-collapsible-content>Singularity CE - Containers for HPC from sylabs.io</div></div></td><td></td></tr><tr><td>skopeo</td><td>1.18.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for skopeo</div><div class=sc-collapsible-content>Skopeo is a command line utility that performs various operations on container images and image repositories</div></div><a href=https://github.com/containers/skopeo/ class=is-pretty-link>https://github.com/containers/skopeo/</a></td><td></td></tr><tr><td>snp-sites</td><td>2.5.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for snp-sites</div><div class=sc-collapsible-content>Description: Extracting single nucleotide polymorphisms (SNPs) from a large whole genome alignment is now a routine task, but existing tools have failed to scale efficiently with the increased size of studies</div></div><a href=https://github.com/sanger-pathogens/snp-sites class=is-pretty-link>https://github.com/sanger-pathogens/snp-sites</a></td><td></td></tr><tr><td>snpeff</td><td>5.2f</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for snpeff</div><div class=sc-collapsible-content>Description: SnpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of genetic variants (such as amino acid changes)</div></div><a href=https://pcingola.github.io/SnpEff/ class=is-pretty-link>https://pcingola.github.io/SnpEff/</a></td><td></td></tr><tr><td>sparsehash</td><td>2.0.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for sparsehash</div><div class=sc-collapsible-content>Description: An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library contains several hash-map implementations, including implementations that optimize for space or speed</div></div><a href=https://github.com/sparsehash/sparsehash class=is-pretty-link>https://github.com/sparsehash/sparsehash</a></td><td></td></tr><tr><td>sqlite</td><td>3.35.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for sqlite</div><div class=sc-collapsible-content>Description: SQLite3 is an SQL database engine in a C library</div></div><a href=https://www.sqlite.org class=is-pretty-link>https://www.sqlite.org</a></td><td></td></tr><tr><td>stata</td><td>15.0-fagfs, 18.0-ffin</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for stata</div><div class=sc-collapsible-content>Description: Stata 18.0 - Statistics / Data Analysis</div></div><a href=http://www.stata.com class=is-pretty-link>http://www.stata.com</a></td><td></td></tr><tr><td>stringtie</td><td>3.0.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for stringtie</div><div class=sc-collapsible-content>Description: StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts</div></div><a href=https://ccb.jhu.edu/software/stringtie/ class=is-pretty-link>https://ccb.jhu.edu/software/stringtie/</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>structure</td><td>2.3.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for structure</div><div class=sc-collapsible-content>Description: The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed</div></div><a href=http://web.stanford.edu/group/pritchardlab/software/ class=is-pretty-link>http://web.stanford.edu/group/pritchardlab/software/</a></td><td></td></tr><tr><td>subread</td><td>2.0.8</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for subread</div><div class=sc-collapsible-content>Description: High performance read alignment, quantification and mutation discovery</div></div><a href=http://subread.sourceforge.net/ class=is-pretty-link>http://subread.sourceforge.net/</a></td><td></td></tr><tr><td>subversion</td><td>1.14.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for subversion</div><div class=sc-collapsible-content>Description: Apache Subversion - an open source version control system</div></div><a href=https://subversion.apache.org/ class=is-pretty-link>https://subversion.apache.org/</a></td><td></td></tr><tr><td>superlu</td><td>5.3.0, 5.3.0+mkl-2019.5, 7.0.0, 7.0.0+mkl-2019.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for superlu</div><div class=sc-collapsible-content>Description: SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines</div></div><a href=http://crd-legacy.lbl.gov/~xiaoye/SuperLU/ class=is-pretty-link>http://crd-legacy.lbl.gov/~xiaoye/SuperLU/</a></td><td>( arch/avx512 intel/2023.2 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>swig</td><td>4.2.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for swig</div><div class=sc-collapsible-content>Description: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages</div></div><a href=http://www.swig.org class=is-pretty-link>http://www.swig.org</a></td><td></td></tr><tr><td>tbb</td><td>2021.13.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for tbb</div><div class=sc-collapsible-content>Description: Threading Building Blocks (TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability</div></div><a href=https://github.com/oneapi-src/oneTBB class=is-pretty-link>https://github.com/oneapi-src/oneTBB</a></td><td>( arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/9.5.0 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>tblite</td><td>0.4.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for tblite</div><div class=sc-collapsible-content>Description: Light-weight tight-binding framework</div></div><a href=https://github.com/tblite/tblite class=is-pretty-link>https://github.com/tblite/tblite</a></td><td>( arch/avx512 intel/2023.2 )</td></tr><tr><td>trimmomatic</td><td>0.39</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for trimmomatic</div><div class=sc-collapsible-content>Description: Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data. The selection of trimming steps and their associated parameters are supplied on the command line</div></div><a href="http://www.usadellab.org/cms/?page=trimmomatic" class=is-pretty-link>http://www.usadellab.org/cms/?page=trimmomatic</a></td><td></td></tr><tr><td>udunits</td><td>2.2.28</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for udunits</div><div class=sc-collapsible-content>Description: A C library for units of physical quantities and a unit-definition and value-conversion utility</div></div><a href=http://www.unidata.ucar.edu/software/udunits class=is-pretty-link>http://www.unidata.ucar.edu/software/udunits</a></td><td>( arch/avx2 gcc/13.2.0 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>unrar</td><td>7.10.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for unrar</div><div class=sc-collapsible-content>Description: Utility used to extract RAR file archives</div></div><a href=https://www.rarlab.com class=is-pretty-link>https://www.rarlab.com</a></td><td></td></tr><tr><td>valgrind</td><td>3.24.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for valgrind</div><div class=sc-collapsible-content>Description: An instrumentation framework for building dynamic analysis</div></div><a href=http://valgrind.org/ class=is-pretty-link>http://valgrind.org/</a></td><td>( arch/avx512 gcc/9.5.0 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>vasp</td><td>6.1.2-sol, 6.3.2-vtst, 6.5.0-iyer, 6.3.2-vtst-sol</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for vasp</div><div class=sc-collapsible-content>The Vienna Ab initio Simulation Package (VASP)</div></div></td><td>( arch/avx512 intel/2023.2 intelmpi/2021.10 ), ( arch/avx512 gcc/11.5.0 openmpi/4.1.6 ), ( arch/avx512 gcc/13.2.0 openmpi/4.1.6 )</td></tr><tr><td>vaspkit</td><td>1.5.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for vaspkit</div><div class=sc-collapsible-content>Description: VASPKIT aims at providing a powerful and user-friendly interface to perform high throughput analysis of various material properties from the raw calculated data using the widely-used VASP code</div></div><a href=https://vaspkit.com/index.html class=is-pretty-link>https://vaspkit.com/index.html</a></td><td></td></tr><tr><td>vcftools</td><td>0.1.16</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for vcftools</div><div class=sc-collapsible-content>Description: A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project</div></div><a href=https://vcftools.github.io class=is-pretty-link>https://vcftools.github.io</a></td><td>( arch/avx512 )</td></tr><tr><td>velvet</td><td>1.2.10</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for velvet</div><div class=sc-collapsible-content>Description: Sequence assembler for very short reads</div></div><a href=http://www.ebi.ac.uk/~zerbino/velvet/ class=is-pretty-link>http://www.ebi.ac.uk/~zerbino/velvet/</a></td><td></td></tr><tr><td>vep</td><td>113.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for vep</div><div class=sc-collapsible-content>Description: Variant Effect Predictor (VEP) determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently used routines in VEP</div></div><a href=https://www.ensembl.org/info/docs/tools/vep class=is-pretty-link>https://www.ensembl.org/info/docs/tools/vep</a></td><td></td></tr><tr><td>vim</td><td>9.1</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for vim</div><div class=sc-collapsible-content>Description: Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as vi with most UNIX systems and with Apple OS X</div></div><a href=https://www.vim.org class=is-pretty-link>https://www.vim.org</a></td><td></td></tr><tr><td>visit</td><td>3.4.2</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for visit</div><div class=sc-collapsible-content>Description: Open Source, interactive, scalable, visualization, animation and analysis tool</div></div><a href=https://wci.llnl.gov/simulation/computer-codes/visit class=is-pretty-link>https://wci.llnl.gov/simulation/computer-codes/visit</a></td><td></td></tr><tr><td>voroplusplus</td><td>0.4.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for voroplusplus</div><div class=sc-collapsible-content>Description: Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation</div></div><a href=http://math.lbl.gov/voro++/ class=is-pretty-link>http://math.lbl.gov/voro++/</a></td><td></td></tr><tr><td>vtk</td><td>9.4.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for vtk</div><div class=sc-collapsible-content>Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation</div></div><a href=http://www.vtk.org class=is-pretty-link>http://www.vtk.org</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>wine</td><td>10.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for wine</div><div class=sc-collapsible-content>Description: Wine (originally an acronym for &lsquo;Wine Is Not an Emulator&rsquo;) is a compatibility layer capable of running Windows applications on several POSIX-compliant operating systems, such as Linux, macOS, & BSD</div></div><a href=https://www.winehq.org/ class=is-pretty-link>https://www.winehq.org/</a></td><td></td></tr><tr><td>xfemm</td><td>4.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for xfemm</div><div class=sc-collapsible-content>Description: FEMM: Magnetics, Electrostatics, Heat Flow, and Current Flow</div></div><a href=https://www.femm.info/wiki/HomePage class=is-pretty-link>https://www.femm.info/wiki/HomePage</a></td><td>( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>xtb</td><td>6.7.1, 6.7.1+mkl-2019.5</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for xtb</div><div class=sc-collapsible-content>Description: xtb - An extended tight-binding semi-empirical program package</div></div><a href=https://xtb-docs.readthedocs.io class=is-pretty-link>https://xtb-docs.readthedocs.io</a></td><td>( arch/avx512 intel/2023.2 ), ( arch/avx512 gcc/13.2.0 )</td></tr><tr><td>xz</td><td>5.6.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for xz</div><div class=sc-collapsible-content>XZ Utils provide a general-purpose data-compression library and command-line tools for xz and lzma formats</div></div></td><td></td></tr><tr><td>yaml-cpp</td><td>0.8.0</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for yaml-cpp</div><div class=sc-collapsible-content>Description: A YAML parser and emitter in C++ matching the YAML 1.2 spec</div></div><a href=https://github.com/jbeder/yaml-cpp class=is-pretty-link>https://github.com/jbeder/yaml-cpp</a></td><td></td></tr><tr><td>yaxt</td><td>0.11.3</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for yaxt</div><div class=sc-collapsible-content>Description: Yet Another eXchange Tool</div></div><a href=https://www.dkrz.de/redmine/projects/yaxt class=is-pretty-link>https://www.dkrz.de/redmine/projects/yaxt</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 openmpi/4.1.6 )</td></tr><tr><td>z3</td><td>4.13.4</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for z3</div><div class=sc-collapsible-content>Description: Z3 is a theorem prover from Microsoft Research with support for bitvectors, booleans, arrays, floating point numbers, strings, and other data types. This module includes z3-solver, the Python interface of Z3</div></div><a href=https://github.com/Z3Prover/z3 class=is-pretty-link>https://github.com/Z3Prover/z3</a></td><td></td></tr><tr><td>zoltan</td><td>3.83, 3.901</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for zoltan</div><div class=sc-collapsible-content>Description: Zoltan Dynamic Load Balancing and Graph Algorithm Toolkit</div></div><a href=https://sandialabs.github.io/Zoltan/ class=is-pretty-link>https://sandialabs.github.io/Zoltan/</a></td><td>( arch/avx512 gcc/13.2.0 openmpi/4.1.6 ), ( arch/avx512 intel/2023.2 openmpi/4.1.6 )</td></tr><tr><td>zstd</td><td>1.5.6</td><td><div class=sc-collapsible-container><div class=sc-collapsible-header>click to expand WhatIs for zstd</div><div class=sc-collapsible-content>Description: Zstandard, or zstd as short version, is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios</div></div><a href=https://facebook.github.io/zstd/ class=is-pretty-link>https://facebook.github.io/zstd/</a></td><td></td></tr></tbody></table></div><div class="column is-sticky is-paddingless" id=tocWrapper><div class=is-paddingless id=tocContainer><div class=toc-header><h6 class="toc-title is-marginless">On this page:</h6><i title="Back to top" id=tocBackToTop class="fa-solid fa-circle-arrow-up"></i></div><div id=tocContent><div id=toc><nav id=TableOfContents><ul><li><a href=#software-available-on-the-ccenv-software-stack-on-grex>Software available on the CCEnv software stack on Grex.</a></li><li><a href=#software-available-on-the-sbenv-software-stack-on-grex>Software available on the SBEnv software stack on Grex.</a></li></ul></nav></div><div id=taxonomies><div class=taxonomy-wrapper><div class=toc-header><h6 class="toc-title is-marginless">Categories</h6></div><a href=/grex-docs/categories/software/ class=taxonomy><i class="fa-solid fa-circle-dot"></i>Software</a><a href=/grex-docs/categories/applications/ class=taxonomy><i class="fa-solid fa-circle-dot"></i>Applications</a></div><div class=taxonomy-wrapper></div></div></div></div><div class=is-paddingless id=tocCollapsible title="Collapse/Uncollapse table of contents"><i class="fa-solid fa-angles-right fa-lg"></i></div></div></div></div><script type=text/javascript>const baseUrl="https://um-grex.github.io/grex-docs/",codeCopyBefore="Copy to clipboard",codeCopyAfter="Copied to clipboard",svgDownloadLabel="Download as SVG",helperLoadingLabel="Loading",searchNoResults="No result found",introNextLabel="Next",introPrevLabel="Previous",introSkipLabel="✗",introDoneLabel="Done",printLabel="Print current page",qrCodeLabel="Show current page QR code",naCommonLabel="Not available"</script><script type=module src=/grex-docs/js/theme/modules/const.min.js></script><script type=module src=/grex-docs/js/theme/modules/helpers.min.js></script><script type=module src=/grex-docs/js/theme/modules/helpersGlobal.min.js></script><script type=module src=/grex-docs/js/theme/init.min.js></script><script type=module src=/grex-docs/js/theme/navbar.min.js></script><script type=module src=/grex-docs/js/theme/print.min.js></script><script type=module src=/grex-docs/js/theme/qrcode.min.js></script><script type=module src=/grex-docs/js/theme/search.min.js></script><script type=module src=/grex-docs/js/theme/shortcuts.min.js></script><script type=module src=/grex-docs/js/theme/sidebar.min.js></script><script type=module src=/grex-docs/js/theme/toc.min.js></script><script type=module src=/grex-docs/js/shortcodes/collapsible.min.js></script><script type=text/javascript src=/grex-docs/js/theme/browserCompatibility.min.js></script><script type=text/javascript src=/grex-docs/js/external/flexsearch/flexsearch.bundle.min.js></script><script type=text/javascript src=/grex-docs/js/external/qrious/qrious.min.js></script><script type=text/javascript src=/grex-docs/js/external/overlay-scrollbars/overlayscrollbars.min.js></script></body></html>