
---
weight: 1990
linkTitle: "List of Software"
title: "List of Software in SBEnv environment"
description: "List of SBEnv"
categories: ["Software", "Applications"]
#tags: ["Configuration"]
---

## Software available on the CCEnv software stack on Grex.

Grex receives the ComputeCanada/Alliance software environment via CVMFS. Full autogenerated list of software is available at the [Available Software page](https://docs.alliancecan.ca/wiki/Available_software) on the Allince Documentation Wiki.
> Note that Grex does not install local, commercial software licensed only to the National/Alliance systems. The software listed under "List of globally-installed modules" is available. 

## Software available on the SBEnv software stack on Grex.

Below is the auto-generated list of software modules available on Grex's local SBEnv software stack. 
> Note that many items on this list, most notably for commercial software, are available only to the licensed software owners/research groups of that software! Please use __module spider__ , __module help__ and __module load__ commands to check if a particular software module can be loaded by your group.

The list is updated periodically. The most authoritative method to check for available software is still __module spider__ as described on the Using Modules page.

| Software | Versions | Description/Homepage | Module dependencies |
| -------- | -------- | -------------------- | ------------------- |
| CCEnv |  | {{<collapsible title="Expand CCEnv description">}}Description: Compute Canada software modules. Must be loaded to see Compute Canada modules in 'module spider'{{</collapsible>}} [https://docs.computecanada.ca/wiki/Available_software](https://docs.computecanada.ca/wiki/Available_software) |  |
| SBEnv |  | {{<collapsible title="Expand SBEnv description">}}Alma8 software environment for Grex usin Niagara SB{{</collapsible>}} |  |
| adf | 2019.305-impi, 2021.106-impi, 2021.107-impi, 2023.104-impi, 2024.105-impi, 2024.105-impi-aocl, 2025.104-impi-aocl | {{<collapsible title="Expand adf description">}}AMS, Amsterdam Density Functional suite of codes Georg Schrecnekbach group at UManitoba, IntelMPI version{{</collapsible>}} |  |
| admixture | 1.3.0 | {{<collapsible title="Expand admixture description">}}ADMIXTURE: Fast ancestry estimation{{</collapsible>}} |  |
| amber | 24 | {{<collapsible title="Expand amber description">}}Description: Amber (originally Assisted Model Building with Energy Refinement) is software for performing molecular dynamics and structure prediction{{</collapsible>}} [https://ambermd.org/](https://ambermd.org/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| ansys | 2023R2, 2025R2, 21.1 | {{<collapsible title="Expand ansys description">}}ANSYS suite of engineering simulation software, ANSYS suite of engineering simulation software{{</collapsible>}} [http://www.ansys.com](http://www.ansys.com) |  |
| ant | 1.10.15 | {{<collapsible title="Expand ant description">}}Description: Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications{{</collapsible>}} [https://ant.apache.org/](https://ant.apache.org/) |  |
| aocc | 4.2.0, 5.0.0 | {{<collapsible title="Expand aocc description">}}Description: The AOCC compiler system is a high performance, production quality code generation tool{{</collapsible>}} [https://developer.amd.com/amd-aocc/](https://developer.amd.com/amd-aocc/) | (arch/avx2), (arch/avx512) |
| aocl | 4.2.0, 4.2.0-64 | {{<collapsible title="Expand aocl description">}}Description: AMD Optimizing CPU Libraries (AOCL) are a set of numerical libraries optimized for AMD EPYC processor family{{</collapsible>}} [https://developer.amd.com/amd-aocl/](https://developer.amd.com/amd-aocl/) | (arch/avx2 gcc/13.2.0), (arch/avx512 aocc/4.2.0), (arch/avx512 gcc/13.2.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| arch | avx2, avx512 | {{<collapsible title="Expand arch description">}}Description: a module to pick the CPU architecture{{</collapsible>}} | (cuda/11.8.0), (cuda/12.2.2), (cuda/12.4.1) |
| armadillo | 11.4.3, 14.2.2 | {{<collapsible title="Expand armadillo description">}}; Description: Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards a good balance between speed and ease of use. Integer, floating point and complex numbers are supported, as well as a subset of trigonometric and statistics functions{{</collapsible>}} [http://arma.sourceforge.net](http://arma.sourceforge.net) | (arch/avx512 gcc/13.2.0) |
| arpack-ng | 3.9.1+mkl-2019.5, 3.9.1+mkl-2024.1 | {{<collapsible title="Expand arpack-ng description">}}Description: ARPACK is a collection of Fortran77 subroutines designed to solve large scale eigenvalue problems{{</collapsible>}} [https://github.com/opencollab/arpack-ng](https://github.com/opencollab/arpack-ng) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 gcc/13.2.0) |
| arrow | 18.1.0 | {{<collapsible title="Expand arrow description">}}Description: Apache Arrow is a cross-language development platform for in-memory data{{</collapsible>}} [https://arrow.apache.org](https://arrow.apache.org) | (arch/avx512 gcc/13.2.0) |
| asynch | 1.4.3, git-a5d1d77 | {{<collapsible title="Expand asynch description">}}Asynch is numerical library for solving differential equations with a tree structure{{</collapsible>}} | (arch/avx512 gcc/13.2.0 openmpi/5.0.6) |
| autodock | 4.2.6 | {{<collapsible title="Expand autodock description">}}Description: AutoDock is a suite of automated docking tools. It is designed to predict how small molecules, such as substrates or drug candidates, bind to a receptor of known 3D structure{{</collapsible>}} [http://autodock.scripps.edu/](http://autodock.scripps.edu/) | (arch/avx512 gcc/13.2.0), (arch/avx512 intel/2023.2) |
| autodock-vina | 1.2.7 | {{<collapsible title="Expand autodock-vina description">}}Description: AutoDock Vina is an open-source program for doing molecular docking{{</collapsible>}} [http://vina.scripps.edu/index.html](http://vina.scripps.edu/index.html) | (arch/avx512 gcc/13.2.0) |
| autotools | 2022a | {{<collapsible title="Expand autotools description">}}Description: GNU autoconf, automake, gettext, and libtool{{</collapsible>}} [https://www.gnu.org/software/{autoconf,automake,gettext,libtool}/](https://www.gnu.org/software/{autoconf,automake,gettext,libtool}/) | (arch/avx2), (arch/avx512) |
| bamtools | 2.5.2 | {{<collapsible title="Expand bamtools description">}}Description: BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files{{</collapsible>}} [https://github.com/pezmaster31/bamtools](https://github.com/pezmaster31/bamtools) |  |
| beagle | 5.4-20241029 | {{<collapsible title="Expand beagle description">}}Description: Beagle is a software package for phasing genotypes and for imputing ungenotyped markers{{</collapsible>}} [https://faculty.washington.edu/browning/beagle/beagle.html](https://faculty.washington.edu/browning/beagle/beagle.html) |  |
| bedtools | 2.31.1 | {{<collapsible title="Expand bedtools description">}}Description: a swiss-army knife of tools for a wide-range of genomics analysis tasks{{</collapsible>}} [https://github.com/arq5x/bedtools2](https://github.com/arq5x/bedtools2) | (arch/avx512) |
| binutils | 2.42 | {{<collapsible title="Expand binutils description">}}{{</collapsible>}} | (arch/avx512) |
| birch | 3.90, 4.0 | {{<collapsible title="Expand birch description">}}BIRCH is a powerful framework for organizing  and using bioinformatics. ; BIRCH comes with hundreds of commonly-used tools, and allows you to customize your BIRCH site by adding new ones{{</collapsible>}} [https://home.cc.umanitoba.ca/~psgendb/index.html](https://home.cc.umanitoba.ca/~psgendb/index.html) |  |
| blastplus | 2.16.0, 2.17.0 | {{<collapsible title="Expand blastplus description">}}Description: Basic Local Alignment Search Tool, or BLAST, is an algorithm for comparing primary biological sequence information, such as the amino-acid sequences of different proteins or the nucleotides of DNA sequences{{</collapsible>}} [https://blast.ncbi.nlm.nih.gov/](https://blast.ncbi.nlm.nih.gov/) | (arch/avx512 gcc/13.2.0) |
| blat | 3.7 | {{<collapsible title="Expand blat description">}}Description: BLAT on DNA is designed to quickly find sequences of 95% and greater similarity of length 25 bases or more{{</collapsible>}} [https://genome.ucsc.edu/goldenPath/help/blatSpec.html](https://genome.ucsc.edu/goldenPath/help/blatSpec.html) | (arch/avx512 gcc/13.2.0) |
| blis | 0.9.0 | {{<collapsible title="Expand blis description">}}Description: BLIS is a portable software framework for instantiating high-performance BLAS-like dense linear algebra libraries{{</collapsible>}} [https://github.com/flame/blis](https://github.com/flame/blis) | (arch/avx2 gcc/13.2.0), (arch/avx512 gcc/13.2.0) |
| boost | 1.78.0, 1.85.0 | {{<collapsible title="Expand boost description">}}Description: Boost provides free peer-reviewed portable C++ source libraries, emphasizing libraries that work well with the C++ Standard Library{{</collapsible>}} [http://www.boost.org](http://www.boost.org) | (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel/2023.2), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| bowtie2 | 2.5.4 | {{<collapsible title="Expand bowtie2 description">}}Description: Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences{{</collapsible>}} [http://bowtie-bio.sourceforge.net/bowtie2/](http://bowtie-bio.sourceforge.net/bowtie2/) | (arch/avx512) |
| buildah | 1.42.2 | {{<collapsible title="Expand buildah description">}}Buildah is a tool that facilitates building Open Container Initiative (OCI) container images{{</collapsible>}} [https://buildah.io/](https://buildah.io/) |  |
| busco | 5.8.3 | {{<collapsible title="Expand busco description">}}Description: BUSCO provides Assessment of assembly completeness using Universal Single Copy Orthologs{{</collapsible>}} [https://busco.ezlab.org](https://busco.ezlab.org) |  |
| bwa | 0.7.18 | {{<collapsible title="Expand bwa description">}}Description: BWA - Burrows-Wheeler Aligner is an efficient program that aligns relatively short nucleotide sequences{{</collapsible>}} [http://bio-bwa.sourceforge.net/](http://bio-bwa.sourceforge.net/) | (arch/avx512 gcc/13.2.0) |
| cdo | 2.5.0 | {{<collapsible title="Expand cdo description">}}Description: CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data{{</collapsible>}} [https://code.mpimet.mpg.de/projects/cdo](https://code.mpimet.mpg.de/projects/cdo) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| cfitsio | 4.4.1, 4.5.0 | {{<collapsible title="Expand cfitsio description">}}Description: CFITSIO is a library of C and Fortran subroutines for reading and writing data files in FITS (Flexible Image Transport System) data format{{</collapsible>}} [http://heasarc.gsfc.nasa.gov/fitsio/](http://heasarc.gsfc.nasa.gov/fitsio/) |  |
| cgal | 5.5, 6.0.1 | {{<collapsible title="Expand cgal description">}}Description: The Computational Geometry Algorithms Library (CGAL) is a C++ library ; that aims to provide easy access to efficient and reliable algorithms in ; computational geometry{{</collapsible>}} [https://www.cgal.org/](https://www.cgal.org/) | (arch/avx512 gcc/13.2.0) |
| circos | 0.69-9 | {{<collapsible title="Expand circos description">}}Description: Circos is a software package for visualizing data and information. It visualizes data in a circular layout - this makes Circos ideal for exploring relationships between objects or positions{{</collapsible>}} [http://www.circos.ca/](http://www.circos.ca/) | (arch/avx512) |
| cistem | 1.0.0 | {{<collapsible title="Expand cistem description">}}Description: cisTEM is user-friendly software to process cryo-EM images of macromolecular complexes and obtain high-resolution 3D reconstructions from them{{</collapsible>}} [https://cistem.org/](https://cistem.org/) | (arch/avx512 gcc/13.2.0) |
| clhep | 2.4.7.1 | {{<collapsible title="Expand clhep description">}}Description: The CLHEP project is intended to be a set of HEP-specific foundation and utility classes such as random generators, physics vectors, geometry and linear algebra. CLHEP is structured in a set of packages independent of any external package{{</collapsible>}} [https://proj-clhep.web.cern.ch/proj-clhep/](https://proj-clhep.web.cern.ch/proj-clhep/) | (arch/avx512 gcc/13.2.0) |
| cmake | 3.28.4, 3.31.1, 4.1.1 | {{<collapsible title="Expand cmake description">}}Description: A cross-platform, open-source build system{{</collapsible>}} [https://www.cmake.org](https://www.cmake.org) |  |
| code-server | 4.106.3 | {{<collapsible title="Expand code-server description">}}Run VS Code on any machine anywhere and access it in the browser{{</collapsible>}} |  |
| compleasm | 0.2.6 | {{<collapsible title="Expand compleasm description">}}Description: Compleasm: a faster and more accurate reimplementation of BUSCO, a genomics tool{{</collapsible>}} [https://github.com/huangnengCSU/compleasm](https://github.com/huangnengCSU/compleasm) |  |
| cp2k | 2024.3 | {{<collapsible title="Expand cp2k description">}}Description: CP2K is a freely available (GPL) program, written in Fortran 95, to perform atomistic and molecular simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and classical pair and many-body potentials{{</collapsible>}} [https://www.cp2k.org/](https://www.cp2k.org/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| cppzmq | 4.10.0 | {{<collapsible title="Expand cppzmq description">}}Description: cppzmq is a C++ binding for libzmq{{</collapsible>}} [https://github.com/zeromq/cppzmq](https://github.com/zeromq/cppzmq) |  |
| crest | 2.12 | {{<collapsible title="Expand crest description">}}Description: CREST is an utility/driver program for the xtb program. Originally it was designed as conformer sampling program, hence the abbreviation Conformer-Rotamer Ensemble Sampling Tool, but now offers also some utility functions for calculations with the GFNn-xTB methods. Generally the program functions as an IO based OMP scheduler (i.e., calculations are performed by the xtb program) and tool for the creation and analysation of structure ensembles{{</collapsible>}} [https://xtb-docs.readthedocs.io/en/latest/crest.html](https://xtb-docs.readthedocs.io/en/latest/crest.html) | (arch/avx512 intel/2023.2) |
| ctffind | 4.1.14 | {{<collapsible title="Expand ctffind description">}}CTFFIND4 is a fast and accurate defocus estimation from electron micrographs{{</collapsible>}} [https://grigoriefflab.umassmed.edu/ctffind4](https://grigoriefflab.umassmed.edu/ctffind4) | (arch/avx512 intel-one/2024.1), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| cuda | 11.8.0, 12.2.2, 12.4.1 | {{<collapsible title="Expand cuda description">}}Description: CUDA is a parallel computing platform and programming model invented by NVIDIA{{</collapsible>}} [https://developer.nvidia.com/cuda-zone](https://developer.nvidia.com/cuda-zone) | (arch/avx2), (arch/avx512) |
| cudnn | 8.8.1.3+cuda-11.8.0 | {{<collapsible title="Expand cudnn description">}}Description: NVIDIA cuDNN is a GPU-accelerated library of primitives for deep neural networks{{</collapsible>}} [https://developer.nvidia.com/cudnn](https://developer.nvidia.com/cudnn) |  |
| deepvariant | 1.8.0, 1.8.0-gpu | {{<collapsible title="Expand deepvariant description">}}DeepVariant is a deep learning-based variant caller that takes aligned reads (in BAM or CRAM format), produces pileup image tensors from them, classifies each tensor using a convolutional neural network, and finally reports the results in a standard VCF or gVCF file{{</collapsible>}} |  |
| dejagnu | 1.6.3 | {{<collapsible title="Expand dejagnu description">}}Description: DejaGnu is a framework for testing other programs{{</collapsible>}} [https://www.gnu.org/software/dejagnu/](https://www.gnu.org/software/dejagnu/) |  |
| diamond | 2.1.10 | {{<collapsible title="Expand diamond description">}}Description: Accelerated BLAST compatible local sequence aligner{{</collapsible>}} [https://github.com/bbuchfink/diamond](https://github.com/bbuchfink/diamond) |  |
| dlb | 3.5.0 | {{<collapsible title="Expand dlb description">}}Description: DLB is a dynamic library designed to speed up HPC hybrid applications by improving the load balance of the outer level of parallelism by dynamically redistributing the computational resources at the inner level of parallelism at run time{{</collapsible>}} [https://pm.bsc.es/dlb/](https://pm.bsc.es/dlb/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| eccodes | 2.31.0, 2.40.0 | {{<collapsible title="Expand eccodes description">}}Description: ecCodes is a package developed by ECMWF which provides an application programming interface and a set of tools for decoding and encoding messages in the following formats: WMO FM-92 GRIB edition 1 and edition 2, WMO FM-94 BUFR edition 3 and edition 4, WMO GTS abbreviated header (only decoding){{</collapsible>}} [https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home](https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home) | (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0) |
| eigen | 3.4.0 | {{<collapsible title="Expand eigen description">}}; Description: Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms{{</collapsible>}} [https://eigen.tuxfamily.org/index.php?title=Main_Page](https://eigen.tuxfamily.org/index.php?title=Main_Page) | (arch/avx512), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| esmf | 8.7.0 | {{<collapsible title="Expand esmf description">}}Description: The Earth System Modeling Framework (ESMF) is software for building and coupling weather, climate, and related models{{</collapsible>}} [http://earthsystemmodeling.org](http://earthsystemmodeling.org) | (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| espresso | 7.3.1, 7.3.1+aocl-4.2.0, 7.4.1, 7.4.1+aocl-4.2.0, 7.5, 7.5+aocl-4.2.0 | {{<collapsible title="Expand espresso description">}}Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. ;  Description: Quantum ESPRESSO  is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft ;  Description: Quantum ESPRESSO is an integrated suite of computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials (both norm-conserving and ultrasoft){{</collapsible>}} [https://www.quantum-espresso.org,](https://www.quantum-espresso.org,) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| expect | 5.45.4 | {{<collapsible title="Expand expect description">}}Description: Expect is a tool for automating interactive applications such as telnet, ftp, passwd, fsck, rlogin, tip, etc{{</collapsible>}} [http://expect.sourceforge.net/](http://expect.sourceforge.net/) |  |
| fastqc | 0.12.1 | {{<collapsible title="Expand fastqc description">}}Description: FastQC is a quality control application for high throughput sequence data. It reads in sequence data in a variety of formats and can either provide an interactive application to review the results of several different QC checks, or create an HTML based report which can be integrated into a pipeline{{</collapsible>}} [https://www.bioinformatics.babraham.ac.uk/projects/fastqc/](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) |  |
| fasttree | 2.1.11 | {{<collapsible title="Expand fasttree description">}}Description: FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences{{</collapsible>}} [http://www.microbesonline.org/fasttree/](http://www.microbesonline.org/fasttree/) | (arch/avx512 gcc/13.2.0), (arch/avx512 intel-one/2024.1) |
| feko | 2021.2 | {{<collapsible title="Expand feko description">}}Altair Feko high-frequency electromagnetic simulation software for Okhmatovsky group{{</collapsible>}} |  |
| ffmpeg | 7.0.2 | {{<collapsible title="Expand ffmpeg description">}}A statically build binary for ffmpeg Linux{{</collapsible>}} [https://johnvansickle.com/ffmpeg/](https://johnvansickle.com/ffmpeg/) |  |
| fftw | 3.3.10 | {{<collapsible title="Expand fftw description">}}Description: FFTW is a C subroutine library for computing the discrete Fourier transform (DFT){{</collapsible>}} [http://www.fftw.org](http://www.fftw.org) | (arch/avx512 aocc/4.2.0 openmpi/4.1.6), (arch/avx512 aocc/4.2.0), (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/14.3.0), (arch/avx512 gcc/9.5.0 openmpi/4.1.6), (arch/avx512 intel-one/2024.1 openmpi/4.1.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel-one/2024.1), (arch/avx512 intel/2023.2 openmpi/4.1.6), (arch/avx512 intel/2023.2), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| flex | 2.6.4 | {{<collapsible title="Expand flex description">}}Description: Flex is a tool for generating scanners{{</collapsible>}} [https://github.com/westes/flex](https://github.com/westes/flex) |  |
| flexpart | 11 | {{<collapsible title="Expand flexpart description">}}Description: FLEXPART (FLEXible PARTicle dispersion model) is a Lagrangian transport and dispersion model suitable for the simulation of a large range of atmospheric transport processes{{</collapsible>}} [https://www.flexpart.eu](https://www.flexpart.eu) | (arch/avx512 gcc/13.2.0) |
| freeglut | 3.4.0 | {{<collapsible title="Expand freeglut description">}}Description: freeglut is a free-software/open-source alternative to the OpenGL Utility Toolkit (GLUT) library{{</collapsible>}} [https://freeglut.sourceforge.net/](https://freeglut.sourceforge.net/) |  |
| gate | 9.4 | {{<collapsible title="Expand gate description">}}Description: GATE is an advanced opensource software developed by the international OpenGATE collaboration and dedicated to the numerical simulations in medical imaging. It currently supports simulations of Emission Tomography (Positron Emission Tomography - PET and Single Photon Emission Computed Tomography - SPECT), and Computed Tomography{{</collapsible>}} [http://www.opengatecollaboration.org/](http://www.opengatecollaboration.org/) | (arch/avx512 gcc/13.2.0) |
| gatk | 4.6.1.0 | {{<collapsible title="Expand gatk description">}}Description: The Genome Analysis Toolkit or GATK is a software package developed at the Broad Institute to analyse next-generation resequencing data. The toolkit offers a wide variety of tools, with a primary focus on variant discovery and genotyping as well as strong emphasis on data quality assurance. Its robust architecture, powerful processing engine and high-performance computing features make it capable of taking on projects of any size{{</collapsible>}} [http://www.broadinstitute.org/gatk/](http://www.broadinstitute.org/gatk/) | (arch/avx512 gcc/13.2.0) |
| gaussian | g16.c01 | {{<collapsible title="Expand gaussian description">}}Gaussian 16.c01 - A quantum chemistry package{{</collapsible>}} |  |
| gcc | 11.5.0, 12.3.0, 13.2.0, 14.3.0, 8.5.0, 9.5.0 | {{<collapsible title="Expand gcc description">}}Description: The GNU Compiler Collection for C, C++, and Fortran{{</collapsible>}} [https://gcc.gnu.org](https://gcc.gnu.org) | (arch/avx2), (arch/avx512), (cuda/11.8.0 arch/avx2), (cuda/12.2.2 arch/avx2), (cuda/12.4.1 arch/avx2) |
| gdal | 3.10.0, 3.9.1 | {{<collapsible title="Expand gdal description">}}Description: GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for data translation and processing{{</collapsible>}} [https://www.gdal.org/](https://www.gdal.org/) | (arch/avx512 gcc/13.2.0) |
| geant4 | 11.2.2, 11.3.0 | {{<collapsible title="Expand geant4 description">}}Description: Geant4 is a toolkit for the simulation of the passage of particles through matter. Its areas of application include high energy, nuclear and accelerator physics, as well as studies in medical and space science{{</collapsible>}} [http://geant4.cern.ch/](http://geant4.cern.ch/) | (arch/avx512 gcc/13.2.0) |
| geos | 3.13.0 | {{<collapsible title="Expand geos description">}}Description: GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS){{</collapsible>}} [https://trac.osgeo.org/geos](https://trac.osgeo.org/geos) | (arch/avx512 gcc/13.2.0) |
| gibbs2 | 1.0 | {{<collapsible title="Expand gibbs2 description">}}Description: Gibbs2 is a program for the calculation of thermodynamic properties in periodic solids under arbitrary conditions of temperature and pressure{{</collapsible>}} [https://aoterodelaroza.github.io/gibbs2](https://aoterodelaroza.github.io/gibbs2) | (arch/avx512 gcc/13.2.0) |
| git | 2.51.0, 2.52.0 | {{<collapsible title="Expand git description">}}Description: Git is a free and open source distributed version control system{{</collapsible>}} [http://git-scm.com](http://git-scm.com) |  |
| git-annex | 10.20251114 | {{<collapsible title="Expand git-annex description">}}Git-annex allows managing large files with git, without storing the file contents in git{{</collapsible>}} [https://git-annex.branchable.com/](https://git-annex.branchable.com/) |  |
| git-lfs | 3.7.0, 3.7.1 | {{<collapsible title="Expand git-lfs description">}}Git Large File Storage (git-LFS) is a command line extension and specification for managing large files with Git{{</collapsible>}} [https://git-lfs.com/](https://git-lfs.com/) |  |
| globus | 3.38.0, 3.40.0 | {{<collapsible title="Expand globus description">}}Tools to interact with Globus using the command line interface: ; - Globus CLI (https://github.com/globus/globus-cli) ; - Globus Connect Personal (https://www.globus.org/globus-connect-personal){{</collapsible>}} [https://www.globus.org/](https://www.globus.org/) |  |
| glost | 0.3.1 | {{<collapsible title="Expand glost description">}}Description: This is GLOST, the Greedy Launcher Of Small Tasks{{</collapsible>}} [https://github.com/cea-hpc/glost](https://github.com/cea-hpc/glost) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| glpk | 5.0 | {{<collapsible title="Expand glpk description">}}Description: The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems. It is a set of routines written in ANSI C and organized in the form of a callable library{{</collapsible>}} [https://www.gnu.org/software/glpk/](https://www.gnu.org/software/glpk/) | (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel/2023.2) |
| gmp | 6.2.1, 6.3.0 | {{<collapsible title="Expand gmp description">}}Description: GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating point numbers{{</collapsible>}} [https://gmplib.org/](https://gmplib.org/) | (arch/avx512 gcc/13.2.0), (arch/avx512), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| gnina | 1.1, 2024 | {{<collapsible title="Expand gnina description">}}gnina (pronounced NEE-na) is a molecular docking program with integrated support for scoring and optimizing ligands using convolutional neural networks{{</collapsible>}} [https://github.com/gnina/gnina](https://github.com/gnina/gnina) |  |
| gnuplot | 5.4.2, 6.0.2 | {{<collapsible title="Expand gnuplot description">}}Description: Gnuplot is a portable command-line driven graphing utility for Linux, OS/2, MS Windows, OSX, VMS, and many other platforms{{</collapsible>}} [http://www.gnuplot.info](http://www.gnuplot.info) |  |
| golang | 1.25.4, 1.25.5 | {{<collapsible title="Expand golang description">}}Go is an opensource programming language supported by Google{{</collapsible>}} |  |
| grace | 5.99.0 | {{<collapsible title="Expand grace description">}}Description: Grace is a WYSIWYG 2D plotting tool for X Windows System and Motif{{</collapsible>}} [http://freecode.com/projects/grace](http://freecode.com/projects/grace) | (arch/avx512 gcc/13.2.0) |
| gromacs | 2021.6, 2022, 2023.3, 2024.1, 2025.2, 2025.3 | {{<collapsible title="Expand gromacs description">}}Description: GROMACS (GROningen MAchine for Chemical Simulations) is a molecular dynamics package primarily designed for simulations of proteins, lipids and nucleic acids{{</collapsible>}} [http://www.gromacs.org](http://www.gromacs.org) | (arch/avx512 aocc/4.2.0 openmpi/4.1.6), (arch/avx512 gcc/11.5.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| gsl | 2.7 | {{<collapsible title="Expand gsl description">}}Description: The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers{{</collapsible>}} [http://www.gnu.org/software/gsl](http://www.gnu.org/software/gsl) | (arch/avx2 gcc/13.2.0), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel-one/2024.1), (arch/avx512 intel/2023.2), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| haploview | 4.2 | {{<collapsible title="Expand haploview description">}}Description: Haploview is designed to simplify and expedite the process of haplotype analysis by providing a common interface to several tasks relating to such analyses{{</collapsible>}} [https://www.broadinstitute.org/haploview/haploview](https://www.broadinstitute.org/haploview/haploview) |  |
| hdf5 | 1.10.9, 1.12.3, 1.14.2, 1.14.6 | {{<collapsible title="Expand hdf5 description">}}Description: HDF5 is a data model, library, and file format for storing and managing data{{</collapsible>}} [https://portal.hdfgroup.org](https://portal.hdfgroup.org) | (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/14.3.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel-one/2024.1 openmpi/4.1.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel-one/2024.1), (arch/avx512 intel/2023.2 intelmpi/2021.10), (arch/avx512 intel/2023.2 openmpi/4.1.6), (arch/avx512 intel/2023.2), (cuda/12.4.1 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| highfive | 2.10.1 | {{<collapsible title="Expand highfive description">}}Description: HighFive is a modern header-only C++11 friendly interface for libhdf5{{</collapsible>}} [https://github.com/BlueBrain/HighFive](https://github.com/BlueBrain/HighFive) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| hisat2 | 2.2.1 | {{<collapsible title="Expand hisat2 description">}}Description: HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) against the general human population (as well as against a single reference genome){{</collapsible>}} [https://daehwankimlab.github.io/hisat2/](https://daehwankimlab.github.io/hisat2/) | (arch/avx512 gcc/13.2.0) |
| hmmer | 3.4 | {{<collapsible title="Expand hmmer description">}}Description: HMMER is used for searching sequence databases for homologs of protein sequences, and for making protein sequence alignments. It implements methods using probabilistic models called profile hidden Markov models (profile HMMs){{</collapsible>}} [http://hmmer.org/](http://hmmer.org/) |  |
| homer | 5.1 | {{<collapsible title="Expand homer description">}}Description: HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and ; next-gen sequencing analysis.  It is a collection of command line programs for unix-style operating systems written ; in Perl and C++. HOMER was primarily written as a de novo motif discovery algorithm and is well suited for finding ; 8-20 bp motifs in large scale genomics data.  HOMER contains many useful tools for analyzing ChIP-Seq, GRO-Seq, ; RNA-Seq, DNase-Seq, Hi-C and numerous other types of functional genomics sequencing data sets{{</collapsible>}} [http://homer.ucsd.edu/homer/download.html](http://homer.ucsd.edu/homer/download.html) | (arch/avx512 gcc/13.2.0) |
| htgettoken | 2.0-2 | {{<collapsible title="Expand htgettoken description">}}htgettoken gets OIDC bearer tokens by interacting with Hashicorp vault{{</collapsible>}} [https://github.com/fermitools/htgettoken](https://github.com/fermitools/htgettoken) |  |
| imb | 2021.7 | {{<collapsible title="Expand imb description">}}; IMB is Intel suite for MPI benchmarks. ; This module only includes classic C benchmarks that can be built by most of compilers. ;  ;  ;, ; IMB is Intel suite for MPI benchmarks. ; This module only includes classic C benchmarks that can be built by most of compilers{{</collapsible>}} [https://github.com/intel/mpi-benchmarks](https://github.com/intel/mpi-benchmarks) | (arch/avx2 intel/2023.2 intelmpi/2021.10), (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel/2023.2 intelmpi/2021.10) |
| intel | 2019.5, 2023.2 | {{<collapsible title="Expand intel description">}}; Description: Intel compilers for C, C++, and Fortran ;   ; Description: Intel compilers for C, C++, and Fortran{{</collapsible>}} [https://software.intel.com/content/www/us/en/develop/tools/oneapi.html](https://software.intel.com/content/www/us/en/develop/tools/oneapi.html) | (arch/avx2), (arch/avx512) |
| intel-one | 2023.2, 2024.1, 2025.3 | {{<collapsible title="Expand intel-one description">}}; Description: Intel compilers for C, C++, and Fortran from New OneAPI suite ;   ;  ; Description: Intel compilers for C, C++, and Fortran from New OneAPI suite{{</collapsible>}} [https://software.intel.com/content/www/us/en/develop/tools/oneapi.html](https://software.intel.com/content/www/us/en/develop/tools/oneapi.html) | (arch/avx512) |
| intelmpi | 2019.8, 2021.10 | {{<collapsible title="Expand intelmpi description">}}Description: Intel MPI library with compiler wrappers for C, C++, and Fortran{{</collapsible>}} [https://software.intel.com/en-us/intel-mpi-library](https://software.intel.com/en-us/intel-mpi-library) | (arch/avx2 gcc/13.2.0), (arch/avx2 intel/2023.2), (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0), (arch/avx512 intel-one/2024.1), (arch/avx512 intel/2023.2) |
| jags | 4.3.2+mkl-2019.5, 4.3.2+mkl-2024.1 | {{<collapsible title="Expand jags description">}}Description: JAGS is Just Another Gibbs Sampler. It is a program for analysis of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation{{</collapsible>}} [http://mcmc-jags.sourceforge.net/](http://mcmc-jags.sourceforge.net/) | (arch/avx512 gcc/13.2.0) |
| jasper | 4.0.0 | {{<collapsible title="Expand jasper description">}}Description: The JasPer Project is an open-source initiative to provide a free software-based reference implementation of the codec specified in the JPEG-2000 Part-1 standard{{</collapsible>}} [https://www.ece.uvic.ca/~frodo/jasper/](https://www.ece.uvic.ca/~frodo/jasper/) | (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0) |
| jellyfish | 2.3.1 | {{<collapsible title="Expand jellyfish description">}}Description: Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA{{</collapsible>}} [http://www.genome.umd.edu/jellyfish.html](http://www.genome.umd.edu/jellyfish.html) |  |
| jemalloc | 5.3.0 | {{<collapsible title="Expand jemalloc description">}}Description: jemalloc is a general purpose malloc(3) implementation that emphasizes fragmentation avoidance and scalable concurrency support{{</collapsible>}} [http://jemalloc.net](http://jemalloc.net) |  |
| jq | 1.7, 1.8.1 | {{<collapsible title="Expand jq description">}}Description: jq is a lightweight and flexible command-line JSON processor{{</collapsible>}} [https://stedolan.github.io/jq/](https://stedolan.github.io/jq/) |  |
| julia | 1.10.3, 1.11.3 | {{<collapsible title="Expand julia description">}}Description: The Julia Language: A fresh approach to technical computing{{</collapsible>}} [http://julialang.org](http://julialang.org) |  |
| kim | 2.3.0 | {{<collapsible title="Expand kim description">}}Open Knowledgebase of Interatomic Models (KIM){{</collapsible>}} | (arch/avx2 gcc/13.2.0), (arch/avx512 gcc/13.2.0), (arch/avx512 intel-one/2024.1), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| kitops | 1.6.0 | {{<collapsible title="Expand kitops description">}}Packaging for AI/ML projects{{</collapsible>}} |  |
| kokkos | 4.4.01 | {{<collapsible title="Expand kokkos description">}}KOKKOS is a portable, hardware agnostic layer for writing C++ HPC applications{{</collapsible>}} | (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| lammps | 2021-09-29, 2024-08-29p1, 2024-08-29p1-nep | {{<collapsible title="Expand lammps description">}}Description: LAMMPS stands for Large-scale Atomic/Molecular Massively Parallel Simulator{{</collapsible>}} [http://www.lammps.org/](http://www.lammps.org/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 intel-one/2024.1 openmpi/4.1.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6) |
| libaec | 1.0.6 | {{<collapsible title="Expand libaec description">}}Description: Libaec provides fast lossless compression of 1 up to 32 bit wide signed or unsigned integers (samples). The library achieves best results for low entropy data as often encountered in space imaging instrument data or numerical model output from weather or climate simulations. While floating point representations are not directly supported, they can also be efficiently coded by grouping exponents and mantissa{{</collapsible>}} [https://gitlab.dkrz.de/k202009/libaec](https://gitlab.dkrz.de/k202009/libaec) |  |
| libint-cp2k | 2.6 | {{<collapsible title="Expand libint-cp2k description">}}Description: Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory{{</collapsible>}} [https://github.com/evaleev/libint](https://github.com/evaleev/libint) | (arch/avx512 gcc/13.2.0) |
| libvori | 220621 | {{<collapsible title="Expand libvori description">}}libvori is a Library for Voronoi Integration written in C++{{</collapsible>}} |  |
| libxc | 5.1.5, 6.2.2 | {{<collapsible title="Expand libxc description">}}Description: Libxc is a library of exchange-correlation functionals for density-functional theory. The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals{{</collapsible>}} [https://www.tddft.org/programs/octopus/wiki/index.php/Libxc](https://www.tddft.org/programs/octopus/wiki/index.php/Libxc) | (arch/avx512 aocc/4.2.0), (arch/avx512 gcc/13.2.0), (arch/avx512 intel-one/2024.1), (arch/avx512 intel/2023.2) |
| libxml2 | 2.11.9 | {{<collapsible title="Expand libxml2 description">}}Description: libxml2 is an XML toolkit implemented in C, originally developed for the GNOME Project{{</collapsible>}} [https://gitlab.gnome.org/GNOME/libxml2](https://gitlab.gnome.org/GNOME/libxml2) |  |
| mathematica | 14.2 | {{<collapsible title="Expand mathematica description">}}Wolfram Mathematica symbolic algebra package (U. of Manitoba users ONLY){{</collapsible>}} [https://www.wolfram.com/mathematica/](https://www.wolfram.com/mathematica/) |  |
| matlab | R2020B2, R2022A, R2023B, R2024A, R2025A | {{<collapsible title="Expand matlab description">}}MATLAB, a computing environment and a programming language from Mathworks. ;  MATLAB, a computing environment from Mathworks{{</collapsible>}} [https://www.mathworks.com/products/matlab.html,](https://www.mathworks.com/products/matlab.html,) |  |
| matlab-proxy | 0.27.1, 0.29.0 | {{<collapsible title="Expand matlab-proxy description">}}Description: matlab-proxy is a Python® package which enables you to launch MATLAB® and access it from a web browser{{</collapsible>}} [https://github.com/mathworks/matlab-proxy](https://github.com/mathworks/matlab-proxy) |  |
| mcr | R2020b, R2022a, R2023b, R2024a, R2025a | {{<collapsible title="Expand mcr description">}}Description: Run MATLAB applications on computers that do not have MATLAB installed{{</collapsible>}} [https://www.mathworks.com/help/compiler/matlab-runtime.html](https://www.mathworks.com/help/compiler/matlab-runtime.html) |  |
| megahit | 1.2.9 | {{<collapsible title="Expand megahit description">}}Description: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph{{</collapsible>}} [https://github.com/voutcn/megahit](https://github.com/voutcn/megahit) |  |
| metashape-pro | 2.1.4, 2.2.1, 2.2.2 | {{<collapsible title="Expand metashape-pro description">}}Agisoft Metashape is a stand-alone software product that performs photogrammetric processing of digital images ; and generates 3D spatial data to be used in GIS applications, cultural heritage documentation, ; and visual effects production as well as for indirect measurements of objects of various scales. ;  ;  Software to perform photogrammetric processing of digital images, generating 3D spatial data{{</collapsible>}} [https://www.agisoft.com/,](https://www.agisoft.com/,) |  |
| metis | 5.1.0 | {{<collapsible title="Expand metis description">}}Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices{{</collapsible>}} [http://glaros.dtc.umn.edu/gkhome/metis/metis/overview](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview) | (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel/2023.2) |
| metis32 | 5.1.0 | {{<collapsible title="Expand metis32 description">}}Description: METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices{{</collapsible>}} [http://glaros.dtc.umn.edu/gkhome/metis/metis/overview](http://glaros.dtc.umn.edu/gkhome/metis/metis/overview) | (arch/avx512 gcc/13.2.0) |
| micro | 2.0.14 | {{<collapsible title="Expand micro description">}}micro is a  simple, modern and intuitive text editor{{</collapsible>}} |  |
| mii | 1.1.1 | {{<collapsible title="Expand mii description">}}Description: A smart search engine for module environments{{</collapsible>}} [https://github.com/codeandkey/mii](https://github.com/codeandkey/mii) |  |
| minimap2 | 2.28 | {{<collapsible title="Expand minimap2 description">}}Minimap2 is a versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database{{</collapsible>}} |  |
| mkl | 2019.5, 2024.1 | {{<collapsible title="Expand mkl description">}}Description: Intel Math Kernel Library{{</collapsible>}} [https://software.intel.com/en-us/mkl](https://software.intel.com/en-us/mkl) | (arch/avx2 gcc/13.2.0), (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0), (cuda/11.8.0 arch/avx2 gcc/11.5.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| molden | 7.3 | {{<collapsible title="Expand molden description">}}Description: Molden is a package for displaying Molecular Density from the Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical packages Mopac/Ampac{{</collapsible>}} [http://www.cmbi.ru.nl/molden/](http://www.cmbi.ru.nl/molden/) |  |
| mpfr | 4.2.1 | {{<collapsible title="Expand mpfr description">}}Description: The MPFR library is a C library for multiple-precision ; floating-point computations with correct rounding{{</collapsible>}} [https://www.mpfr.org/](https://www.mpfr.org/) | (arch/avx512 gcc/13.2.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| mumax3 | 3.10 | {{<collapsible title="Expand mumax3 description">}}Mumax3 GPU-accelerated micromagnetic simulation program for CUDA{{</collapsible>}} |  |
| mummer | 4.0.0rc1 | {{<collapsible title="Expand mummer description">}}Description: MUMmer is a system for rapidly aligning entire genomes, whether in complete or draft form. AMOS makes use of it{{</collapsible>}} [http://mummer.sourceforge.net/](http://mummer.sourceforge.net/) |  |
| mustang | 3.2.4 | {{<collapsible title="Expand mustang description">}}Description: MUSTANG (MUltiple STructural AligNment AlGorithm), for the alignment of multiple protein structures{{</collapsible>}} [http://www.csse.monash.edu.au/~karun/Site/mustang.html](http://www.csse.monash.edu.au/~karun/Site/mustang.html) | (arch/avx512 gcc/13.2.0) |
| nbo | nbo7-2021 | {{<collapsible title="Expand nbo description">}}The Natural Bond Orbital (NBO) program NBO 7{{</collapsible>}} |  |
| nco | 5.3.1 | {{<collapsible title="Expand nco description">}}Description: The NCO toolkit manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, HDF5, and, most recently, Zarr{{</collapsible>}} [https://nco.sourceforge.net](https://nco.sourceforge.net) | (arch/avx512 gcc/13.2.0) |
| ncview | 2.1.11 | {{<collapsible title="Expand ncview description">}}Description: Ncview is a visual browser for netCDF format files. Typically you would use ncview to get a quick and easy, push-button ; look at your netCDF files. You can view simple movies of the data, view along various dimensions, take a look at the actual data values, change color maps, invert the data, etc{{</collapsible>}} [http://meteora.ucsd.edu/~pierce/ncview_home_page.html](http://meteora.ucsd.edu/~pierce/ncview_home_page.html) | (arch/avx512 gcc/13.2.0) |
| netcdf | 4.9.2+hdf5-1.14.2 | {{<collapsible title="Expand netcdf description">}}Description: NetCDF (network Common Data Form) is a set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data{{</collapsible>}} [http://www.unidata.ucar.edu/software/netcdf](http://www.unidata.ucar.edu/software/netcdf) | (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0), (arch/avx512 intel-one/2024.1), (arch/avx512 intel/2023.2), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| nextflow | 24.10.0 | {{<collapsible title="Expand nextflow description">}}Nextflow is a reactive workflow framework and a programming DSL that eases writing computational pipelines with complex data{{</collapsible>}} [https://www.nextflow.io/](https://www.nextflow.io/) |  |
| ninja | 1.10.2 | {{<collapsible title="Expand ninja description">}}Description: Ninja is a small build system with a focus on speed{{</collapsible>}} [https://ninja-build.org](https://ninja-build.org) |  |
| nodejs | 18.20.2, 18.20.4, 18.20.5, 20.12.2, 20.15.1, 20.16.0, 20.18.1, 22.11.0, 22.4.1, 22.5.1 | {{<collapsible title="Expand nodejs description">}}Node.JS -  the asynchronous server-side JavaScript language. https://nodejs.org{{</collapsible>}} |  |
| nvtop | 3.1.0, 3.2.0 | {{<collapsible title="Expand nvtop description">}}Description: NVTOP stands for Neat Videocard TOP, a (h)top like task monitor for GPUs and accelerators. It can handle multiple GPUs and print information about them in a htop-familiar way{{</collapsible>}} [https://github.com/Syllo/nvtop](https://github.com/Syllo/nvtop) |  |
| nwchem | 7.2.2, 7.2.2+aocl-4.2.0-64, 7.3.0, 7.3.0+aocl-4.2.0-64 | {{<collapsible title="Expand nwchem description">}}Description: NWChem is an open source high performance computing package for computational chemistry. ; Homepage:  https://nwchemgit.github.io/index.html, Description: NWChem is an open source high performance computing package for computational chemistry{{</collapsible>}} | (arch/avx512 aocc/4.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/4.1.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6) |
| omlmd | 0.1.6 | {{<collapsible title="Expand omlmd description">}}OCI Artifact for ML model and metadata{{</collapsible>}} |  |
| openbabel | 3.1.1 | {{<collapsible title="Expand openbabel description">}}Description: Open Babel is a chemical toolbox designed to speak the many languages of chemical data. It's an open, collaborative project allowing anyone to search, convert, analyze, or store data from molecular modeling, chemistry, solid-state materials, biochemistry, or related areas{{</collapsible>}} [https://openbabel.org/](https://openbabel.org/) | (arch/avx512 gcc/13.2.0) |
| openblas | 0.3.26, 0.3.28 | {{<collapsible title="Expand openblas description">}}Description: OpenBLAS: An optimized BLAS library{{</collapsible>}} [https://www.openblas.net](https://www.openblas.net) | (arch/avx512 aocc/4.2.0), (arch/avx512 gcc/13.2.0), (arch/avx512 intel/2023.2), (arch/avx512), (cuda/11.8.0 arch/avx2 gcc/11.5.0), (cuda/12.4.1 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| openeye | 2022.2.1 | {{<collapsible title="Expand openeye description">}}OpenEye: module for R. Davis Group only{{</collapsible>}} |  |
| openfoam | 9 | {{<collapsible title="Expand openfoam description">}}Description: OpenFOAM is a GPL-opensource C++ CFD-toolbox{{</collapsible>}} [https://www.openfoam.com/](https://www.openfoam.com/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6) |
| openjdk | 11.0.22, 17.0.11_9, 17.0.12_7, 17.0.13_11, 21.0.2, 21.0.3_9, 21.0.4_7, 21.0.5_11, 21.0.6_7 | {{<collapsible title="Expand openjdk description">}}Description: The free and opensource java implementation{{</collapsible>}} [https://jdk.java.net](https://jdk.java.net) |  |
| openmm | 8.2.0, 8.3.1 | {{<collapsible title="Expand openmm description">}}Description: OpenMM is a toolkit for molecular simulation{{</collapsible>}} [http://openmm.org](http://openmm.org) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/4.1.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6) |
| openmpi | 4.1.6, 5.0.6, 5.0.9 | {{<collapsible title="Expand openmpi description">}}Description: An open source Message Passing Interface implementation{{</collapsible>}} [http://www.open-mpi.org](http://www.open-mpi.org) | (arch/avx2 aocc/4.2.0), (arch/avx2 intel/2023.2), (arch/avx512 aocc/4.2.0), (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/14.3.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel-one/2024.1), (arch/avx512 intel-one/2025.3), (arch/avx512 intel/2023.2), (cuda/12.2.2 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| opennurbs | 8.12 | {{<collapsible title="Expand opennurbs description">}}Description: The openNURBS Initiative provides CAD, CAM, CAE, and computer graphics software developers the tools to accurately transfer 3D geometry between applications{{</collapsible>}} [https://github.com/mcneel/opennurbs](https://github.com/mcneel/opennurbs) | (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0) |
| openssl | 3.4.0 | {{<collapsible title="Expand openssl description">}}Description: OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols{{</collapsible>}} [http://www.openssl.org](http://www.openssl.org) |  |
| openstack-client | 8.2.0 | {{<collapsible title="Expand openstack-client description">}}OpenStackClient is a command-line client for OpenStack{{</collapsible>}} |  |
| orca | 5.0.4, 6.0.1, 6.1.0, 6.1.1 | {{<collapsible title="Expand orca description">}}Description: ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum chemistry with specific emphasis on spectroscopic properties of open-shell molecules. ;  Description ; =========== ; ORCA is a flexible, efficient and easy-to-use general purpose tool for quantum ; chemistry with specific emphasis on spectroscopic properties of open-shell molecules. ; It features a wide variety of standard quantum chemical methods ranging from ; semiempirical methods to DFT to single- and multireference correlated ab initio methods. ; It can also treat environmental and relativistic effects{{</collapsible>}} [http://cec.mpg.de/forum/,](http://cec.mpg.de/forum/,) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| osu-micro-benchmarks | 5.7.1 | {{<collapsible title="Expand osu-micro-benchmarks description">}}Description: The Ohio MicroBenchmark suite is a collection of independent MPI message passing performance microbenchmarks developed and written at The Ohio State University{{</collapsible>}} [http://mvapich.cse.ohio-state.edu/benchmarks/](http://mvapich.cse.ohio-state.edu/benchmarks/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| ovito | 3.12.3, 3.13.1, 3.14.0 | {{<collapsible title="Expand ovito description">}}OVITO (Open Visualization Tool) is a scientific data visualization and analysis software designed for molecular and particle-based simulation models. ;  OVITO (Open Visualization Tool) is a scientific data visualization and analysis software designed for molecular and particle-based simulation models{{</collapsible>}} [https://www.ovito.org/,](https://www.ovito.org/,) |  |
| pandaseq | 2.11 | {{<collapsible title="Expand pandaseq description">}}Description: PANDASEQ is a program to align Illumina reads and reconstruct an overlapping sequence{{</collapsible>}} [https://github.com/neufeld/pandaseq](https://github.com/neufeld/pandaseq) | (arch/avx512 gcc/13.2.0) |
| pandoc | 3.6.1 | {{<collapsible title="Expand pandoc description">}}Description: If you need to convert files from one markup format into another, pandoc is your swiss-army knife{{</collapsible>}} [https://pandoc.org](https://pandoc.org) |  |
| paraview | 5.13.3 | {{<collapsible title="Expand paraview description">}}Description: ParaView is a scientific parallel visualizer{{</collapsible>}} [https://www.paraview.org](https://www.paraview.org) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6) |
| paraview-offscreen | 5.10.1 | {{<collapsible title="Expand paraview-offscreen description">}}Description: Off-screen rendering with ParaView, a scientific parallel visualizer{{</collapsible>}} [http://www.paraview.org](http://www.paraview.org) |  |
| parmetis | 4.0.3-shared | {{<collapsible title="Expand parmetis description">}}Description: ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices{{</collapsible>}} [http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview](http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| perl | 5.38.2, 5.40.1 | {{<collapsible title="Expand perl description">}}Description: Perl 5 is a highly capable, feature-rich programming language with over 27 years of development{{</collapsible>}} [http://www.perl.org](http://www.perl.org) | (arch/avx2), (arch/avx512) |
| petsc-complex | 3.13.3+mkl-2019.5, 3.22.2+mkl-2019.5 | {{<collapsible title="Expand petsc-complex description">}}Description: PETSc is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations{{</collapsible>}} [http://www.mcs.anl.gov/petsc/index.html](http://www.mcs.anl.gov/petsc/index.html) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| petsc-real | 3.13.3+mkl-2019.5, 3.22.2+mkl-2019.5 | {{<collapsible title="Expand petsc-real description">}}Description: PETSc is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations{{</collapsible>}} [http://www.mcs.anl.gov/petsc/index.html](http://www.mcs.anl.gov/petsc/index.html) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6) |
| picard | 3.3.0 | {{<collapsible title="Expand picard description">}}Picard: A set of tools (in Java) for working with next generation sequencing data in the BAM format{{</collapsible>}} [https://broadinstitute.github.io/picard/](https://broadinstitute.github.io/picard/) |  |
| pnetcdf | 1.14.0 | {{<collapsible title="Expand pnetcdf description">}}Description: Parallel netCDF: A Parallel I/O Library for NetCDF File Access{{</collapsible>}} [https://trac.mcs.anl.gov/projects/parallel-netcdf](https://trac.mcs.anl.gov/projects/parallel-netcdf) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| podman | 5.6.2, 5.7.0 | {{<collapsible title="Expand podman description">}}Podman (the POD MANager) is a tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers{{</collapsible>}} [https://podman.io/](https://podman.io/) |  |
| podman-compose | 1.5.0 | {{<collapsible title="Expand podman-compose description">}}An implementation of Compose Spec with Podman backend{{</collapsible>}} [https://github.com/containers/podman-compose](https://github.com/containers/podman-compose) |  |
| podman-tui | 1.9.0 | {{<collapsible title="Expand podman-tui description">}}Podman-tui is a terminal user interface for podman environment. ; It uses podman go bindings to communicate with local or remote podman machine (through SSH){{</collapsible>}} [https://github.com/containers/podman-tui/](https://github.com/containers/podman-tui/) |  |
| potreeconv | 2.1.1 | {{<collapsible title="Expand potreeconv description">}}Description: PotreeConverter generates an octree LOD structure for streaming and real-time rendering of massive point clouds{{</collapsible>}} [https://github.com/potree/PotreeConverter](https://github.com/potree/PotreeConverter) | (arch/avx512 gcc/13.2.0) |
| process-compose | 1.78.0 | {{<collapsible title="Expand process-compose description">}}Process Compose is a simple and flexible scheduler and orchestrator to manage non-containerized applications{{</collapsible>}} |  |
| prodigal | 2.6.3 | {{<collapsible title="Expand prodigal description">}}Description: Prodigal (Prokaryotic Dynamic Programming Genefinding Algorithm) is a microbial (bacterial and archaeal) gene finding program developed at Oak Ridge National Laboratory and the University of Tennessee{{</collapsible>}} [https://github.com/hyattpd/Prodigal/](https://github.com/hyattpd/Prodigal/) |  |
| proj | 9.2.0, 9.5.0 | {{<collapsible title="Expand proj description">}}Description: Program proj is a standard Unix filter function which converts geographic longitude and latitude coordinates into cartesian coordinates{{</collapsible>}} [https://proj.org](https://proj.org) | (arch/avx512 gcc/13.2.0) |
| prsice | 2.3.5 | {{<collapsible title="Expand prsice description">}}Description: PRSice (pronounced 'precise') is a software package for calculating, applying, evaluating and plotting the results of polygenic risk scores (PRS). PRSice can run at high-resolution to provide the best-fit PRS as well as provide results calculated at broad P-value thresholds, illustrating results corresponding to either, can thin SNPs according to linkage disequilibrium and P-value ( clumping ), and can be applied across multiple traits in a single run{{</collapsible>}} [https://github.com/choishingwan/PRSice/tree/master](https://github.com/choishingwan/PRSice/tree/master) | (arch/avx512) |
| python | 3.10.14, 3.10.16, 3.11.11, 3.11.8, 3.12.9 | {{<collapsible title="Expand python description">}}Description: The Python programming language{{</collapsible>}} [https://www.python.org/](https://www.python.org/) | (arch/avx2 gcc/13.2.0), (arch/avx512 aocc/4.2.0), (arch/avx512 gcc/11.5.0), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/14.3.0), (arch/avx512 gcc/9.5.0), (arch/avx512 intel-one/2024.1), (cuda/11.8.0 arch/avx2 gcc/11.5.0), (cuda/12.2.2 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| qhull | 2020.2 | {{<collapsible title="Expand qhull description">}}Description: Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point, furthest-site Delaunay triangulation, and furthest-site Voronoi diagram{{</collapsible>}} [http://www.qhull.org](http://www.qhull.org) | (arch/avx512 gcc/13.2.0) |
| qiime2 | 2024.10 | {{<collapsible title="Expand qiime2 description">}}Description: QIIME is an open-source bioinformatics pipeline for performing microbiome analysis from raw DNA sequencing data{{</collapsible>}} [http://qiime2.org/](http://qiime2.org/) |  |
| qrupdate | 1.1.2 | {{<collapsible title="Expand qrupdate description">}}Description: Qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions{{</collapsible>}} [https://sourceforge.net/projects/qrupdate/](https://sourceforge.net/projects/qrupdate/) | (arch/avx512 gcc/13.2.0) |
| qt | 6.7.1, 6.8.1, 6.8.3, 6.9.1 | {{<collapsible title="Expand qt description">}}Description: Qt is a comprehensive cross-platform C++ application framework{{</collapsible>}} [https://qt.io/](https://qt.io/) | (arch/avx512 gcc/13.2.0) |
| quast | 5.3.0 | {{<collapsible title="Expand quast description">}}Description: QUAST is an open-source bioinformatics Quality Assessment Tool for Genome Assemblies{{</collapsible>}} |  |
| r | 4.3.3, 4.4.1+aocl-4.2.0, 4.4.1+mkl-2019.5, 4.4.1+mkl-2024.1, 4.5.0+mkl-2024.1, 4.5.0+openblas-0.3.28 | {{<collapsible title="Expand r description">}}Description: R is a language and environment for statistical computing and graphics{{</collapsible>}} [https://www.r-project.org](https://www.r-project.org) | (arch/avx2 gcc/13.2.0), (arch/avx2), (arch/avx512 gcc/13.2.0), (arch/avx512), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| ramalama | 0.15.0 | {{<collapsible title="Expand ramalama description">}}RamaLama tool facilitates local management and serving of AI Models{{</collapsible>}} [https://github.com/containers/ramalama](https://github.com/containers/ramalama) |  |
| ratarmount | 1.2.1 | {{<collapsible title="Expand ratarmount description">}}Random Access To Archived Resources (Ratarmount) ; Ratarmount collects all file positions inside a TAR so that it can easily jump to and read from any file without extracting it. ; It, then, mounts the TAR using fusepy for read access just like archivemount. ; In contrast to libarchive, on which archivemount is based, random access and true seeking is supported. ; And in contrast to tarindexer, which also collects file positions for random access, ratarmount offers easy access via FUSE and support for compressed TARs{{</collapsible>}} [https://github.com/mxmlnkn/ratarmount](https://github.com/mxmlnkn/ratarmount) |  |
| raxml-ng | 1.2.1 | {{<collapsible title="Expand raxml-ng description">}}Description: RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion{{</collapsible>}} [https://github.com/amkozlov/raxml-ng](https://github.com/amkozlov/raxml-ng) | (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6) |
| rclone | 1.71.0, 1.72.0 | {{<collapsible title="Expand rclone description">}}Rclone: sync files/directories to and from a variety of online storage services{{</collapsible>}} [https://rclone.org/](https://rclone.org/) |  |
| relion | 4.0.2, 5.0.0 | {{<collapsible title="Expand relion description">}}Description: RELION (for REgularised LIkelihood OptimisatioN) is a stand-alone computer program for Maximum. A Posteriori refinement of (multiple) 3D reconstructions or 2D class averages in cryo-electron microscopy{{</collapsible>}} | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6) |
| root | 6.32.08, 6.34.02 | {{<collapsible title="Expand root description">}}Description: The ROOT system provides a set of OO frameworks with all the functionality needed to handle and analyze large amounts of data in a very efficient way{{</collapsible>}} [https://root.cern.ch/drupal/](https://root.cern.ch/drupal/) | (arch/avx512 gcc/13.2.0) |
| rs-server | 2025.09.1-401, 2025.09.2-418 | {{<collapsible title="Expand rs-server description">}}RStudio server for use with Ondemand or Jupyter{{</collapsible>}} |  |
| rust | 1.90.0, 1.91.1 | {{<collapsible title="Expand rust description">}}Description: Rust is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety{{</collapsible>}} [https://www.rust-lang.org](https://www.rust-lang.org) |  |
| s3cmd | 2.4.0 | {{<collapsible title="Expand s3cmd description">}}Free command line tool and client for uploading, retrieving and managing data ; in Amazon S3 and other cloud storage service providers that use the S3 protocol, ; such as Google Cloud Storage{{</collapsible>}} [https://github.com/s3tools/s3cmd](https://github.com/s3tools/s3cmd) |  |
| samtools | 1.20 | {{<collapsible title="Expand samtools description">}}Description: SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format{{</collapsible>}} [http://www.htslib.org](http://www.htslib.org) | (arch/avx512 gcc/13.2.0) |
| scalapack | 2.2.2 | {{<collapsible title="Expand scalapack description">}}Description: The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines redesigned for distributed memory MIMD parallel computers. This module uses OpenBLAS. Please use AOCL or MKL modules for more efficient implementations of ScaLAPACK!{{</collapsible>}} [https://www.netlib.org/scalapack/](https://www.netlib.org/scalapack/) | (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (cuda/12.4.1 arch/avx2 gcc/13.2.0 openmpi/5.0.6) |
| scipy-bundle | 2023+python-3.10.14, 2023+python-3.10.16, 2023+python-3.11.8 | {{<collapsible title="Expand scipy-bundle description">}}A bundle of common Python packages: Numpy, Scipy, Pandas, Matplotlib etc{{</collapsible>}} | (arch/avx512 gcc/13.2.0), (cuda/12.4.1 arch/avx2 gcc/12.3.0), (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| scons | 3.1.2 | {{<collapsible title="Expand scons description">}}Description: SCons is a software construction tool{{</collapsible>}} [http://scons.org](http://scons.org) |  |
| scotch | 6.0.9, 7.0.5 | {{<collapsible title="Expand scotch description">}}Description: Software package and libraries for sequential and parallel graph partitioning, static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning{{</collapsible>}} [https://www.labri.fr/perso/pelegrin/scotch/](https://www.labri.fr/perso/pelegrin/scotch/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| singularity | 4.1.2, 4.2.2, 4.3.4, 4.3.6 | {{<collapsible title="Expand singularity description">}}Singularity CE  - Containers for HPC from sylabs.io{{</collapsible>}} |  |
| skopeo | 1.21.0 | {{<collapsible title="Expand skopeo description">}}Skopeo is a command line utility that performs various operations on container images and image repositories{{</collapsible>}} [https://github.com/containers/skopeo/](https://github.com/containers/skopeo/) |  |
| snp-sites | 2.5.1 | {{<collapsible title="Expand snp-sites description">}}Description: Extracting single nucleotide polymorphisms (SNPs) from a large whole genome alignment is now a routine task, but existing tools have failed to scale efficiently with the increased size of studies{{</collapsible>}} [https://github.com/sanger-pathogens/snp-sites](https://github.com/sanger-pathogens/snp-sites) |  |
| snpeff | 5.2f | {{<collapsible title="Expand snpeff description">}}Description: SnpEff is a variant annotation and effect prediction tool. It annotates and predicts the effects of genetic variants (such as amino acid changes){{</collapsible>}} [https://pcingola.github.io/SnpEff/](https://pcingola.github.io/SnpEff/) |  |
| sparsehash | 2.0.4 | {{<collapsible title="Expand sparsehash description">}}Description: An extremely memory-efficient hash_map implementation. 2 bits/entry overhead! The SparseHash library contains several hash-map implementations, including implementations that optimize for space or speed{{</collapsible>}} [https://github.com/sparsehash/sparsehash](https://github.com/sparsehash/sparsehash) |  |
| sqlite | 3.35.5 | {{<collapsible title="Expand sqlite description">}}Description: SQLite3 is an SQL database engine in a C library{{</collapsible>}} [https://www.sqlite.org](https://www.sqlite.org) |  |
| stata | 15.0-fagfs, 18.0-ffin | {{<collapsible title="Expand stata description">}}Description: Stata 18.0 - Statistics / Data Analysis{{</collapsible>}} [http://www.stata.com](http://www.stata.com) |  |
| stringtie | 3.0.0 | {{<collapsible title="Expand stringtie description">}}Description: StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts{{</collapsible>}} [https://ccb.jhu.edu/software/stringtie/](https://ccb.jhu.edu/software/stringtie/) | (arch/avx512 gcc/13.2.0) |
| structure | 2.3.4 | {{<collapsible title="Expand structure description">}}Description: The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed{{</collapsible>}} [http://web.stanford.edu/group/pritchardlab/software/](http://web.stanford.edu/group/pritchardlab/software/) |  |
| subread | 2.0.8 | {{<collapsible title="Expand subread description">}}Description: High performance read alignment, quantification and mutation discovery{{</collapsible>}} [http://subread.sourceforge.net/](http://subread.sourceforge.net/) |  |
| subversion | 1.14.3 | {{<collapsible title="Expand subversion description">}}Description: Apache Subversion - an open source version control system{{</collapsible>}} [https://subversion.apache.org/](https://subversion.apache.org/) |  |
| sundials | 7.4.0+mkl-2024.1 | {{<collapsible title="Expand sundials description">}}Description: SUNDIALS: SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers{{</collapsible>}} [https://computing.llnl.gov/projects/sundials](https://computing.llnl.gov/projects/sundials) | (arch/avx512 gcc/13.2.0 openmpi/5.0.6) |
| superlu | 5.3.0, 5.3.0+mkl-2019.5, 7.0.0, 7.0.0+mkl-2019.5 | {{<collapsible title="Expand superlu description">}}Description: SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations on high performance machines{{</collapsible>}} [http://crd-legacy.lbl.gov/~xiaoye/SuperLU/](http://crd-legacy.lbl.gov/~xiaoye/SuperLU/) | (arch/avx512 gcc/13.2.0), (arch/avx512 intel/2023.2) |
| swig | 4.2.0 | {{<collapsible title="Expand swig description">}}Description: SWIG is a software development tool that connects programs written in C and C++ with a variety of high-level programming languages{{</collapsible>}} [http://www.swig.org](http://www.swig.org) |  |
| taudem | 5.3.8 | {{<collapsible title="Expand taudem description">}}Description: TauDEM (Terrain Analysis Using Digital Elevation Models) is a suite of Digital Elevation Model (DEM){{</collapsible>}} [https://github.com/dtarb/TauDEM/wiki](https://github.com/dtarb/TauDEM/wiki) | (arch/avx512 gcc/13.2.0 openmpi/5.0.6) |
| tbb | 2021.13.0 | {{<collapsible title="Expand tbb description">}}Description: Threading Building Blocks (TBB) lets you easily write parallel C++ programs that take full advantage of multicore performance, that are portable, composable and have future-proof scalability{{</collapsible>}} [https://github.com/oneapi-src/oneTBB](https://github.com/oneapi-src/oneTBB) | (arch/avx2 gcc/13.2.0), (arch/avx512 gcc/13.2.0), (arch/avx512 gcc/9.5.0) |
| tblite | 0.4.0 | {{<collapsible title="Expand tblite description">}}Description: Light-weight tight-binding framework{{</collapsible>}} [https://github.com/tblite/tblite](https://github.com/tblite/tblite) | (arch/avx512 intel/2023.2) |
| trimmomatic | 0.39 | {{<collapsible title="Expand trimmomatic description">}}Description: Trimmomatic performs a variety of useful trimming tasks for illumina paired-end and single ended data. The selection of trimming steps and their associated parameters are supplied on the command line{{</collapsible>}} [http://www.usadellab.org/cms/?page=trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic) |  |
| udunits | 2.2.28 | {{<collapsible title="Expand udunits description">}}Description: A C library for units of physical quantities and a unit-definition and value-conversion utility{{</collapsible>}} [http://www.unidata.ucar.edu/software/udunits](http://www.unidata.ucar.edu/software/udunits) | (arch/avx2 gcc/13.2.0), (arch/avx512 gcc/13.2.0) |
| unrar | 7.10.2 | {{<collapsible title="Expand unrar description">}}Description: Utility used to extract RAR file archives{{</collapsible>}} [https://www.rarlab.com](https://www.rarlab.com) |  |
| valgrind | 3.24.0 | {{<collapsible title="Expand valgrind description">}}Description: An instrumentation framework for building dynamic analysis{{</collapsible>}} [http://valgrind.org/](http://valgrind.org/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 gcc/9.5.0 openmpi/4.1.6) |
| vasp | 6.1.2-sol, 6.3.2-vtst, 6.3.2-vtst-sol, 6.5.0-iyer, 6.5.1-deng | {{<collapsible title="Expand vasp description">}}The Vienna Ab initio Simulation Package (VASP){{</collapsible>}} | (arch/avx512 gcc/11.5.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 intel/2023.2 intelmpi/2021.10) |
| vaspkit | 1.5.1 | {{<collapsible title="Expand vaspkit description">}}Description: VASPKIT aims at providing a powerful and user-friendly interface to perform high throughput analysis of various material properties from the raw calculated data using the widely-used VASP code{{</collapsible>}} [https://vaspkit.com/index.html](https://vaspkit.com/index.html) |  |
| vcftools | 0.1.16 | {{<collapsible title="Expand vcftools description">}}Description: A set of tools written in Perl and C++ for working with VCF files, such as those generated by the 1000 Genomes Project{{</collapsible>}} [https://vcftools.github.io](https://vcftools.github.io) | (arch/avx512) |
| velvet | 1.2.10 | {{<collapsible title="Expand velvet description">}}Description: Sequence assembler for very short reads{{</collapsible>}} [http://www.ebi.ac.uk/~zerbino/velvet/](http://www.ebi.ac.uk/~zerbino/velvet/) |  |
| vep | 113.4 | {{<collapsible title="Expand vep description">}}Description: Variant Effect Predictor (VEP) determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions. Includes EnsEMBL-XS, which provides pre-compiled replacements for frequently used routines in VEP{{</collapsible>}} [https://www.ensembl.org/info/docs/tools/vep](https://www.ensembl.org/info/docs/tools/vep) |  |
| vim | 9.1 | {{<collapsible title="Expand vim description">}}Description: Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as  vi  with most UNIX systems and with Apple OS X{{</collapsible>}} [https://www.vim.org](https://www.vim.org) |  |
| visit | 3.4.2 | {{<collapsible title="Expand visit description">}}Description: Open Source, interactive, scalable, visualization, animation and analysis tool{{</collapsible>}} [https://wci.llnl.gov/simulation/computer-codes/visit](https://wci.llnl.gov/simulation/computer-codes/visit) |  |
| voroplusplus | 0.4.6 | {{<collapsible title="Expand voroplusplus description">}}Description: Voro++ is a software library for carrying out three-dimensional computations of the Voronoi tessellation{{</collapsible>}} [http://math.lbl.gov/voro++/](http://math.lbl.gov/voro++/) |  |
| vtk | 9.4.0 | {{<collapsible title="Expand vtk description">}}Description: The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation{{</collapsible>}} [http://www.vtk.org](http://www.vtk.org) | (arch/avx512 gcc/13.2.0) |
| wine | 10.0 | {{<collapsible title="Expand wine description">}}Description: Wine (originally an acronym for 'Wine Is Not an Emulator') is a compatibility layer capable of running Windows applications on several POSIX-compliant operating systems, such as Linux, macOS, & BSD{{</collapsible>}} [https://www.winehq.org/](https://www.winehq.org/) |  |
| wxwidgets | 3.0.2 | {{<collapsible title="Expand wxwidgets description">}}Description: wxWidgets is a C++ library that lets developers create applications for Windows, Mac OS X, Linux and other platforms with a ; single code base. It has popular language bindings for Python, Perl, Ruby and many other languages, and unlike other cross-platform toolkits, wxWidgets gives applications a truly native look and feel because it uses the platform's native API rather than emulating the GUI{{</collapsible>}} [https://www.wxwidgets.org](https://www.wxwidgets.org) | (arch/avx512 gcc/13.2.0) |
| xfemm | 4.0 | {{<collapsible title="Expand xfemm description">}}Description: FEMM: Magnetics, Electrostatics, Heat Flow, and Current Flow{{</collapsible>}} [https://www.femm.info/wiki/HomePage](https://www.femm.info/wiki/HomePage) | (arch/avx512 gcc/13.2.0) |
| xgboost | 3.0.2 | {{<collapsible title="Expand xgboost description">}}Description: XGBoost is an open-source software library that implements gradient boosting algorithms{{</collapsible>}} [https://xgboost.readthedocs.io/](https://xgboost.readthedocs.io/) | (cuda/12.4.1 arch/avx2 gcc/13.2.0) |
| xisearch | 1.8.11 | {{<collapsible title="Expand xisearch description">}}Description: xiSEARCH is a search engine for the identification of crosslinked spectra matches in crosslinking mass spectrometry experiments. It is implemented as a Java Application{{</collapsible>}} [https://www.rappsilberlab.org/software/xisearch/](https://www.rappsilberlab.org/software/xisearch/) |  |
| xtb | 6.7.1, 6.7.1+mkl-2019.5 | {{<collapsible title="Expand xtb description">}}Description: xtb - An extended tight-binding semi-empirical program package{{</collapsible>}} [https://xtb-docs.readthedocs.io](https://xtb-docs.readthedocs.io) | (arch/avx512 gcc/13.2.0), (arch/avx512 intel/2023.2) |
| xz | 5.6.4 | {{<collapsible title="Expand xz description">}}XZ Utils provide a general-purpose data-compression library and command-line tools for xz and lzma formats{{</collapsible>}} |  |
| yaml-cpp | 0.8.0 | {{<collapsible title="Expand yaml-cpp description">}}Description: A YAML parser and emitter in C++ matching the YAML 1.2 spec{{</collapsible>}} [https://github.com/jbeder/yaml-cpp](https://github.com/jbeder/yaml-cpp) |  |
| yaxt | 0.11.3 | {{<collapsible title="Expand yaxt description">}}Description: Yet Another eXchange Tool{{</collapsible>}} [https://www.dkrz.de/redmine/projects/yaxt](https://www.dkrz.de/redmine/projects/yaxt) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| z3 | 4.13.4 | {{<collapsible title="Expand z3 description">}}Description: Z3 is a theorem prover from Microsoft Research with support for bitvectors, booleans, arrays, floating point numbers, strings, and other data types. This module includes z3-solver, the Python interface of Z3{{</collapsible>}} [https://github.com/Z3Prover/z3](https://github.com/Z3Prover/z3) |  |
| zoltan | 3.83, 3.901 | {{<collapsible title="Expand zoltan description">}}Description: Zoltan Dynamic Load Balancing and Graph Algorithm Toolkit{{</collapsible>}} [https://sandialabs.github.io/Zoltan/](https://sandialabs.github.io/Zoltan/) | (arch/avx512 gcc/13.2.0 openmpi/4.1.6), (arch/avx512 gcc/13.2.0 openmpi/5.0.6), (arch/avx512 intel-one/2024.1 openmpi/5.0.6), (arch/avx512 intel/2023.2 openmpi/4.1.6) |
| zstd | 1.5.6 | {{<collapsible title="Expand zstd description">}}Description: Zstandard, or zstd as short version, is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios{{</collapsible>}} [https://facebook.github.io/zstd/](https://facebook.github.io/zstd/) |  |


<!-- Changes and update:
* Last reviewed on: 2026-01-01 03:00:04
-->

